
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://airr-knowledge.github.io/ak-schema/SampleProcessing/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-8.5.10" name="generator"/>
<title>Class: SampleProcessing - ak-schema</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-sampleprocessing">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="ak-schema" class="md-header__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            ak-schema
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: SampleProcessing
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="ak-schema" class="md-nav__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    ak-schema
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
    Inheritance
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
    Usages
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/airr-knowledge/ak-schema/edit/master/docs/SampleProcessing.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-sampleprocessing">Class: SampleProcessing</h1>
<p>URI: <a href="https://github.com/airr-knowledge/ak-schema/SampleProcessing">ak_schema:SampleProcessing</a></p>
<div class="mermaid"> classDiagram
    class SampleProcessing
    click SampleProcessing href "../SampleProcessing"
      AIRRStandards &lt;|-- SampleProcessing
        click AIRRStandards href "../AIRRStandards"

      SampleProcessing : anatomic_site

      SampleProcessing : biomaterial_provider

      SampleProcessing : cell_isolation

      SampleProcessing : cell_number

      SampleProcessing : cell_phenotype

      SampleProcessing : cell_processing_protocol

      SampleProcessing : cell_quality

      SampleProcessing : cell_species




    SampleProcessing --&gt; "0..1" CellSpeciesOntology : cell_species
    click CellSpeciesOntology href "../CellSpeciesOntology"


      SampleProcessing : cell_storage

      SampleProcessing : cell_subset




    SampleProcessing --&gt; "0..1" CellSubsetOntology : cell_subset
    click CellSubsetOntology href "../CellSubsetOntology"


      SampleProcessing : cells_per_reaction

      SampleProcessing : collection_time_point_reference

      SampleProcessing : collection_time_point_relative

      SampleProcessing : collection_time_point_relative_unit




    SampleProcessing --&gt; "0..1" CollectionTimePointRelativeUnitOntology : collection_time_point_relative_unit
    click CollectionTimePointRelativeUnitOntology href "../CollectionTimePointRelativeUnitOntology"


      SampleProcessing : complete_sequences




    SampleProcessing --&gt; "0..1" CompleteSequencesEnum : complete_sequences
    click CompleteSequencesEnum href "../CompleteSequencesEnum"


      SampleProcessing : disease_state_sample

      SampleProcessing : library_generation_kit_version

      SampleProcessing : library_generation_method




    SampleProcessing --&gt; "0..1" LibraryGenerationMethodEnum : library_generation_method
    click LibraryGenerationMethodEnum href "../LibraryGenerationMethodEnum"


      SampleProcessing : library_generation_protocol

      SampleProcessing : pcr_target




    SampleProcessing --&gt; "*" PCRTarget : pcr_target
    click PCRTarget href "../PCRTarget"


      SampleProcessing : physical_linkage




    SampleProcessing --&gt; "0..1" PhysicalLinkageEnum : physical_linkage
    click PhysicalLinkageEnum href "../PhysicalLinkageEnum"


      SampleProcessing : sample_id

      SampleProcessing : sample_processing_id

      SampleProcessing : sample_type

      SampleProcessing : sequencing_facility

      SampleProcessing : sequencing_files




    SampleProcessing --&gt; "0..1" SequencingData : sequencing_files
    click SequencingData href "../SequencingData"


      SampleProcessing : sequencing_kit

      SampleProcessing : sequencing_platform

      SampleProcessing : sequencing_run_date

      SampleProcessing : sequencing_run_id

      SampleProcessing : single_cell

      SampleProcessing : template_amount

      SampleProcessing : template_amount_unit




    SampleProcessing --&gt; "0..1" TemplateAmountUnitOntology : template_amount_unit
    click TemplateAmountUnitOntology href "../TemplateAmountUnitOntology"


      SampleProcessing : template_class




    SampleProcessing --&gt; "0..1" TemplateClassEnum : template_class
    click TemplateClassEnum href "../TemplateClassEnum"


      SampleProcessing : template_quality

      SampleProcessing : tissue




    SampleProcessing --&gt; "0..1" TissueOntology : tissue
    click TissueOntology href "../TissueOntology"


      SampleProcessing : tissue_processing

      SampleProcessing : total_reads_passing_qc_filter


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../AIRRStandards/">AIRRStandards</a><ul>
<li><strong>SampleProcessing</strong></li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../sample_processing_id/">sample_processing_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sample_id/">sample_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Sample ID assigned by submitter, unique within study</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sample_type/">sample_type</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The way the sample was obtained, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../tissue/">tissue</a></td>
<td>0..1 <br/> <a href="../TissueOntology/">TissueOntology</a></td>
<td>The actual tissue sampled, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../anatomic_site/">anatomic_site</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The anatomic location of the tissue, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../disease_state_sample/">disease_state_sample</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Histopathologic evaluation of the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../collection_time_point_relative/">collection_time_point_relative</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Time point at which sample was taken, relative to <code>Collection time event</code></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../collection_time_point_relative_unit/">collection_time_point_relative_unit</a></td>
<td>0..1 <br/> <a href="../CollectionTimePointRelativeUnitOntology/">CollectionTimePointRelativeUnitOntology</a></td>
<td>Unit of Sample collection time</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../collection_time_point_reference/">collection_time_point_reference</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Event in the study schedule to which <code>Sample collection time</code> relates to</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../biomaterial_provider/">biomaterial_provider</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name and address of the entity providing the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../tissue_processing/">tissue_processing</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Enzymatic digestion and/or physical methods used to isolate cells from sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_subset/">cell_subset</a></td>
<td>0..1 <br/> <a href="../CellSubsetOntology/">CellSubsetOntology</a></td>
<td>Commonly-used designation of isolated cell population</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_phenotype/">cell_phenotype</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>List of cellular markers and their expression levels used to isolate the cell...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_species/">cell_species</a></td>
<td>0..1 <br/> <a href="../CellSpeciesOntology/">CellSpeciesOntology</a></td>
<td>Binomial designation of the species from which the analyzed cells originate</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../single_cell/">single_cell</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>TRUE if single cells were isolated into separate compartments</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_number/">cell_number</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of cells that went into the experiment</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cells_per_reaction/">cells_per_reaction</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of cells for each biological replicate</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_storage/">cell_storage</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>TRUE if cells were cryo-preserved between isolation and further processing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_quality/">cell_quality</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Relative amount of viable cells after preparation and (if applicable) thawing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_isolation/">cell_isolation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description of the procedure used for marker-based isolation or enrich cells</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../cell_processing_protocol/">cell_processing_protocol</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description of the methods applied to the sample including cell preparation/ ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../template_class/">template_class</a></td>
<td>0..1 <br/> <a href="../TemplateClassEnum/">TemplateClassEnum</a></td>
<td>The class of nucleic acid that was used as primary starting material for the ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../template_quality/">template_quality</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description and results of the quality control performed on the template mate...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../template_amount/">template_amount</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Amount of template that went into the process</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../template_amount_unit/">template_amount_unit</a></td>
<td>0..1 <br/> <a href="../TemplateAmountUnitOntology/">TemplateAmountUnitOntology</a></td>
<td>Unit of template amount</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../library_generation_method/">library_generation_method</a></td>
<td>0..1 <br/> <a href="../LibraryGenerationMethodEnum/">LibraryGenerationMethodEnum</a></td>
<td>Generic type of library generation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../library_generation_protocol/">library_generation_protocol</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Description of processes applied to substrate to obtain a library that is rea...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../library_generation_kit_version/">library_generation_kit_version</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>When using a library generation protocol from a commercial provider, provide ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../pcr_target/">pcr_target</a></td>
<td>* <br/> <a href="../PCRTarget/">PCRTarget</a></td>
<td>If a PCR step was performed that specifically targets the IG/TR loci, the tar...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../complete_sequences/">complete_sequences</a></td>
<td>0..1 <br/> <a href="../CompleteSequencesEnum/">CompleteSequencesEnum</a></td>
<td>To be considered <code>complete</code>, the procedure used for library construction MUST...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../physical_linkage/">physical_linkage</a></td>
<td>0..1 <br/> <a href="../PhysicalLinkageEnum/">PhysicalLinkageEnum</a></td>
<td>In case an experimental setup is used that physically links nucleic acids der...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_run_id/">sequencing_run_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>ID of sequencing run assigned by the sequencing facility</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../total_reads_passing_qc_filter/">total_reads_passing_qc_filter</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of usable reads for analysis</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_platform/">sequencing_platform</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Designation of sequencing instrument used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_facility/">sequencing_facility</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name and address of sequencing facility</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_run_date/">sequencing_run_date</a></td>
<td>0..1 <br/> <a href="../Datetime/">Datetime</a></td>
<td>Date of sequencing run</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_kit/">sequencing_kit</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Name, manufacturer, order and lot numbers of sequencing kit</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequencing_files/">sequencing_files</a></td>
<td>0..1 <br/> <a href="../SequencingData/">SequencingData</a></td>
<td>Set of sequencing files produced by the sequencing run</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../Repertoire/">Repertoire</a></td>
<td><a href="../sample/">sample</a></td>
<td>range</td>
<td><a href="./">SampleProcessing</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://github.com/airr-knowledge/ak-schema</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>ak_schema:SampleProcessing</td>
</tr>
<tr>
<td>native</td>
<td>ak_schema:SampleProcessing</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: SampleProcessing
from_schema: https://github.com/airr-knowledge/ak-schema
is_a: AIRRStandards
slots:
- sample_processing_id
- sample_id
- sample_type
- tissue
- anatomic_site
- disease_state_sample
- collection_time_point_relative
- collection_time_point_relative_unit
- collection_time_point_reference
- biomaterial_provider
- tissue_processing
- cell_subset
- cell_phenotype
- cell_species
- single_cell
- cell_number
- cells_per_reaction
- cell_storage
- cell_quality
- cell_isolation
- cell_processing_protocol
- template_class
- template_quality
- template_amount
- template_amount_unit
- library_generation_method
- library_generation_protocol
- library_generation_kit_version
- pcr_target
- complete_sequences
- physical_linkage
- sequencing_run_id
- total_reads_passing_qc_filter
- sequencing_platform
- sequencing_facility
- sequencing_run_date
- sequencing_kit
- sequencing_files

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: SampleProcessing
from_schema: https://github.com/airr-knowledge/ak-schema
is_a: AIRRStandards
attributes:
  sample_processing_id:
    name: sample_processing_id
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sample_processing_id
    owner: SampleProcessing
    domain_of:
    - Rearrangement
    - SampleProcessing
    range: string
  sample_id:
    name: sample_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Sample ID assigned by submitter, unique within study. If possible,
      a persistent sample ID linked to INSDC or similar repository study should be
      used.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sample_id
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  sample_type:
    name: sample_type
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The way the sample was obtained, e.g. fine-needle aspirate, organ
      harvest, peripheral venous puncture
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    slot_uri: rdf:type
    alias: sample_type
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  tissue:
    name: tissue
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The actual tissue sampled, e.g. lymph node, liver, peripheral blood
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: tissue
    owner: SampleProcessing
    domain_of:
    - Specimen
    - Sample
    - SampleProcessing
    range: TissueOntology
  anatomic_site:
    name: anatomic_site
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The anatomic location of the tissue, e.g. Inguinal, femur
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: anatomic_site
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  disease_state_sample:
    name: disease_state_sample
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Histopathologic evaluation of the sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: disease_state_sample
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  collection_time_point_relative:
    name: collection_time_point_relative
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Time point at which sample was taken, relative to `Collection time
      event`
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: collection_time_point_relative
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: float
  collection_time_point_relative_unit:
    name: collection_time_point_relative_unit
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Unit of Sample collection time
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: collection_time_point_relative_unit
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: CollectionTimePointRelativeUnitOntology
  collection_time_point_reference:
    name: collection_time_point_reference
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Event in the study schedule to which `Sample collection time` relates
      to
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: collection_time_point_reference
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  biomaterial_provider:
    name: biomaterial_provider
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name and address of the entity providing the sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: biomaterial_provider
    owner: SampleProcessing
    domain_of:
    - Sample
    - SampleProcessing
    range: string
  tissue_processing:
    name: tissue_processing
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Enzymatic digestion and/or physical methods used to isolate cells
      from sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: tissue_processing
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: string
  cell_subset:
    name: cell_subset
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Commonly-used designation of isolated cell population
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_subset
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: CellSubsetOntology
  cell_phenotype:
    name: cell_phenotype
    annotations:
      nullable:
        tag: nullable
        value: true
    description: List of cellular markers and their expression levels used to isolate
      the cell population
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_phenotype
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: string
  cell_species:
    name: cell_species
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Binomial designation of the species from which the analyzed cells
      originate. Typically, this value should be identical to `species`, in which
      case it SHOULD NOT be set explicitly. However, there are valid experimental
      setups in which the two might differ, e.g., chimeric animal models. If set,
      this key will overwrite the `species` information for all lower layers of the
      schema.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_species
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: CellSpeciesOntology
  single_cell:
    name: single_cell
    annotations:
      nullable:
        tag: nullable
        value: true
    description: TRUE if single cells were isolated into separate compartments
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: single_cell
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: boolean
  cell_number:
    name: cell_number
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Total number of cells that went into the experiment
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_number
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: integer
  cells_per_reaction:
    name: cells_per_reaction
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Number of cells for each biological replicate
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cells_per_reaction
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: integer
  cell_storage:
    name: cell_storage
    annotations:
      nullable:
        tag: nullable
        value: true
    description: TRUE if cells were cryo-preserved between isolation and further processing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_storage
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: boolean
  cell_quality:
    name: cell_quality
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Relative amount of viable cells after preparation and (if applicable)
      thawing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_quality
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: string
  cell_isolation:
    name: cell_isolation
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of the procedure used for marker-based isolation or enrich
      cells
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_isolation
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: string
  cell_processing_protocol:
    name: cell_processing_protocol
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of the methods applied to the sample including cell preparation/
      isolation/enrichment and nucleic acid extraction. This should closely mirror
      the Materials and methods section in the manuscript.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: cell_processing_protocol
    owner: SampleProcessing
    domain_of:
    - CellIsolationProcessing
    - CellProcessing
    - SampleProcessing
    range: string
  template_class:
    name: template_class
    annotations:
      nullable:
        tag: nullable
        value: false
    description: The class of nucleic acid that was used as primary starting material
      for the following procedures
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: template_class
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: TemplateClassEnum
  template_quality:
    name: template_quality
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description and results of the quality control performed on the template
      material
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: template_quality
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: string
  template_amount:
    name: template_amount
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Amount of template that went into the process
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: template_amount
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: float
  template_amount_unit:
    name: template_amount_unit
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Unit of template amount
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: template_amount_unit
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: TemplateAmountUnitOntology
  library_generation_method:
    name: library_generation_method
    annotations:
      nullable:
        tag: nullable
        value: false
    description: Generic type of library generation
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: library_generation_method
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: LibraryGenerationMethodEnum
  library_generation_protocol:
    name: library_generation_protocol
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of processes applied to substrate to obtain a library
      that is ready for sequencing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: library_generation_protocol
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: string
  library_generation_kit_version:
    name: library_generation_kit_version
    annotations:
      nullable:
        tag: nullable
        value: true
    description: When using a library generation protocol from a commercial provider,
      provide the protocol version number
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: library_generation_kit_version
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: string
  pcr_target:
    name: pcr_target
    annotations:
      nullable:
        tag: nullable
        value: false
    description: If a PCR step was performed that specifically targets the IG/TR loci,
      the target and primer locations need to be provided here. This field holds an
      array of PCRTarget objects, so that multiplex PCR setups amplifying multiple
      loci at the same time can be annotated using one record per locus. PCR setups
      not targeting any specific locus must not annotate this field but select the
      appropriate library_generation_method instead.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: pcr_target
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: PCRTarget
    multivalued: true
  complete_sequences:
    name: complete_sequences
    annotations:
      nullable:
        tag: nullable
        value: false
    description: To be considered `complete`, the procedure used for library construction
      MUST generate sequences that 1) include the first V gene codon that encodes
      the mature polypeptide chain (i.e. after the leader sequence) and 2) include
      the last complete codon of the J gene (i.e. 1 bp 5' of the J-&gt;C splice site)
      and 3) provide sequence information for all positions between 1) and 2). To
      be considered `complete &amp; untemplated`, the sections of the sequences defined
      in points 1) to 3) of the previous sentence MUST be untemplated, i.e. MUST NOT
      overlap with the primers used in library preparation. `mixed` should only be
      used if the procedure used for library construction will likely produce multiple
      categories of sequences in the given experiment. It SHOULD NOT be used as a
      replacement of a NULL value.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: complete_sequences
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: CompleteSequencesEnum
  physical_linkage:
    name: physical_linkage
    annotations:
      nullable:
        tag: nullable
        value: false
    description: In case an experimental setup is used that physically links nucleic
      acids derived from distinct `Rearrangements` before library preparation, this
      field describes the mode of that linkage. All `hetero_*` terms indicate that
      in case of paired-read sequencing, the two reads should be expected to map to
      distinct IG/TR loci. `*_head-head` refers to techniques that link the 5' ends
      of transcripts in a single-cell context. `*_tail-head` refers to techniques
      that link the 3' end of one transcript to the 5' end of another one in a single-cell
      context. This term does not provide any information whether a continuous reading-frame
      between the two is generated. `*_prelinked` refers to constructs in which the
      linkage was already present on the DNA level (e.g. scFv).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: physical_linkage
    owner: SampleProcessing
    domain_of:
    - NucleicAcidProcessing
    - SampleProcessing
    range: PhysicalLinkageEnum
  sequencing_run_id:
    name: sequencing_run_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: ID of sequencing run assigned by the sequencing facility
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_run_id
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: string
  total_reads_passing_qc_filter:
    name: total_reads_passing_qc_filter
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Number of usable reads for analysis
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: total_reads_passing_qc_filter
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: integer
  sequencing_platform:
    name: sequencing_platform
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Designation of sequencing instrument used
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_platform
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: string
  sequencing_facility:
    name: sequencing_facility
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name and address of sequencing facility
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_facility
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: string
  sequencing_run_date:
    name: sequencing_run_date
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Date of sequencing run
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_run_date
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: datetime
  sequencing_kit:
    name: sequencing_kit
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name, manufacturer, order and lot numbers of sequencing kit
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_kit
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: string
  sequencing_files:
    name: sequencing_files
    annotations:
      nullable:
        tag: nullable
        value: false
    description: Set of sequencing files produced by the sequencing run
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequencing_files
    owner: SampleProcessing
    domain_of:
    - SequencingRun
    - SampleProcessing
    range: SequencingData

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script><script type="module">import mermaid from "https://unpkg.com/mermaid@10.9.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>