
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://airr-knowledge.github.io/ak-schema/">
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-8.5.10">
    
    
      
        <title>ak-schema</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.975780f9.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.2505c338.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="None" data-md-color-accent="None">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#airr-knowledge-schema" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="ak-schema" class="md-header__button md-logo" aria-label="ak-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            ak-schema
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Index
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/airr-knowledge/ak-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="ak-schema" class="md-nav__button md-logo" aria-label="ak-schema" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    ak-schema
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/airr-knowledge/ak-schema" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Index
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        Index
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="about/" class="md-nav__link">
        About
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    Classes
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    Enumerations
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    Types
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    Subsets
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/airr-knowledge/ak-schema/edit/master/docs/index.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


<h1 id="airr-knowledge-schema">AIRR Knowledge Schema</h1>
<p>Common data model schema for the AIRR Knowledge Commons</p>
<p>URI: https://github.com/airr-knowledge/ak-schema</p>
<p>Name: ak-schema</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Acknowledgement/">Acknowledgement</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="AIRRKnowledgeCommons/">AIRRKnowledgeCommons</a></td>
<td>A container for instances of multiple classes.</td>
</tr>
<tr>
<td><a href="AKObject/">AKObject</a></td>
<td>Anything uniquely identifiable in the AKC.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="NamedThing/">NamedThing</a></td>
<td>Name and description for AKC things.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="BCellReceptor/">BCellReceptor</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Cell/">Cell</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Chain/">Chain</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Conclusion/">Conclusion</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Dataset/">Dataset</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Epitope/">Epitope</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PeptidicEpitope/">PeptidicEpitope</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ImmuneExposure/">ImmuneExposure</a></td>
<td>An event involving the immune system of a study participant.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ImmuneSystem/">ImmuneSystem</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="LifeEvent/">LifeEvent</a></td>
<td>An event in which a study participant participates.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Model/">Model</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="CommunicativeModel/">CommunicativeModel</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ConceptualModel/">ConceptualModel</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ModelingFramework/">ModelingFramework</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Participant/">Participant</a></td>
<td>A participant in an investigation.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PlannedProcess/">PlannedProcess</a></td>
<td>A process to realize a plan.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Assay/">Assay</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ReceptorRepertoireSequencingAssay/">ReceptorRepertoireSequencingAssay</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TCellReceptorEpitopeBindingAssay/">TCellReceptorEpitopeBindingAssay</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Assessment/">Assessment</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Investigation/">Investigation</a></td>
<td>A scientific investigation.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Simulation/">Simulation</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="SpecimenCollection/">SpecimenCollection</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="SpecimenProcessing/">SpecimenProcessing</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="PlanSpecification/">PlanSpecification</a></td>
<td>A plan with specified actions to meet some objectives.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ModelSpecification/">ModelSpecification</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Specimen/">Specimen</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="StudyArm/">StudyArm</a></td>
<td>A population of participants of an investigation.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="StudyEvent/">StudyEvent</a></td>
<td>An event that is part of the study design of an investigation.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="TCellReceptor/">TCellReceptor</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="AlphaBetaTCR/">AlphaBetaTCR</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="GammaDeltaTCR/">GammaDeltaTCR</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Tissue/">Tissue</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="SimilarityCalculation/">SimilarityCalculation</a></td>
<td>None</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="ChainSimilarity/">ChainSimilarity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Alignment/">Alignment</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="AlleleDescription/">AlleleDescription</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellExpression/">CellExpression</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellProcessing/">CellProcessing</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Clone/">Clone</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DataProcessing/">DataProcessing</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DeletedGene/">DeletedGene</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Diagnosis/">Diagnosis</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DocumentedAllele/">DocumentedAllele</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ForeignObject/">ForeignObject</a></td>
<td>An object held outside of the AK.</td>
</tr>
<tr>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="Reference/">Reference</a></td>
<td>A document about an investigation.</td>
</tr>
<tr>
<td><a href="Genotype/">Genotype</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GenotypeSet/">GenotypeSet</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GermlineSet/">GermlineSet</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="MHCAllele/">MHCAllele</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="MHCGenotype/">MHCGenotype</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="MHCGenotypeSet/">MHCGenotypeSet</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Node/">Node</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="NucleicAcidProcessing/">NucleicAcidProcessing</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="PCRTarget/">PCRTarget</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="RearrangedSequence/">RearrangedSequence</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Rearrangement/">Rearrangement</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Receptor/">Receptor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ReceptorReactivity/">ReceptorReactivity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Repertoire/">Repertoire</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="RepertoireGroup/">RepertoireGroup</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="RepertoireGroupDetail/">RepertoireGroupDetail</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Sample/">Sample</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SampleProcessing/">SampleProcessing</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SequenceDelineationV/">SequenceDelineationV</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SequencingData/">SequencingData</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SequencingRun/">SequencingRun</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Study/">Study</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Subject/">Subject</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SubjectGenotype/">SubjectGenotype</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="TimePoint/">TimePoint</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Tree/">Tree</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="UndocumentedAllele/">UndocumentedAllele</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="UnrearrangedSequence/">UnrearrangedSequence</a></td>
<td>None</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="acknowledgement_id/">acknowledgement_id</a></td>
<td>unique identifier of this Acknowledgement within the file</td>
</tr>
<tr>
<td><a href="acknowledgements/">acknowledgements</a></td>
<td></td>
</tr>
<tr>
<td><a href="adc_publish_date/">adc_publish_date</a></td>
<td>Date the study was first published in the AIRR Data Commons</td>
</tr>
<tr>
<td><a href="adc_update_date/">adc_update_date</a></td>
<td>Date the study data was updated in the AIRR Data Commons</td>
</tr>
<tr>
<td><a href="age/">age</a></td>
<td>The age of a participant relative to age_event</td>
</tr>
<tr>
<td><a href="age_event/">age_event</a></td>
<td>The event to which the age is relative</td>
</tr>
<tr>
<td><a href="age_max/">age_max</a></td>
<td>Upper boundary of age range or equal to age_min for specific age</td>
</tr>
<tr>
<td><a href="age_min/">age_min</a></td>
<td>Specific age or lower boundary of age range</td>
</tr>
<tr>
<td><a href="age_unit/">age_unit</a></td>
<td>Units used to describe the age</td>
</tr>
<tr>
<td><a href="akc_id/">akc_id</a></td>
<td>A unique identifier for a thing in the AKC</td>
</tr>
<tr>
<td><a href="aliases/">aliases</a></td>
<td>Alternative names for this sequence</td>
</tr>
<tr>
<td><a href="aligned_sequence/">aligned_sequence</a></td>
<td>Aligned sequence if this delineation provides an alignment</td>
</tr>
<tr>
<td><a href="alignment_labels/">alignment_labels</a></td>
<td>One string for each codon in the aligned_sequence indicating the label of tha...</td>
</tr>
<tr>
<td><a href="allele_description_id/">allele_description_id</a></td>
<td>Unique identifier of this AlleleDescription within the file</td>
</tr>
<tr>
<td><a href="allele_description_ref/">allele_description_ref</a></td>
<td>Unique reference to the allele description, in standardized form (Repo:Label:...</td>
</tr>
<tr>
<td><a href="allele_descriptions/">allele_descriptions</a></td>
<td>list of allele_descriptions in the germline set</td>
</tr>
<tr>
<td><a href="allele_designation/">allele_designation</a></td>
<td></td>
</tr>
<tr>
<td><a href="allele_name/">allele_name</a></td>
<td>Allele name as allocated by the inference pipeline</td>
</tr>
<tr>
<td><a href="allele_similarity_cluster_designation/">allele_similarity_cluster_designation</a></td>
<td>ID of the similarity cluster used in this germline set, if designated</td>
</tr>
<tr>
<td><a href="allele_similarity_cluster_member_id/">allele_similarity_cluster_member_id</a></td>
<td>Membership ID of the allele within the similarity cluster, if a cluster is de...</td>
</tr>
<tr>
<td><a href="analysis_provenance_id/">analysis_provenance_id</a></td>
<td>Identifier for machine-readable PROV model of analysis provenance</td>
</tr>
<tr>
<td><a href="anatomic_site/">anatomic_site</a></td>
<td>The anatomic location of the tissue, e</td>
</tr>
<tr>
<td><a href="ancestry_population/">ancestry_population</a></td>
<td>Broad geographic origin of ancestry (continent)</td>
</tr>
<tr>
<td><a href="antigen/">antigen</a></td>
<td>The substance against which the receptor was tested</td>
</tr>
<tr>
<td><a href="antigen_source_species/">antigen_source_species</a></td>
<td>The species from which the antigen was isolated</td>
</tr>
<tr>
<td><a href="antigen_type/">antigen_type</a></td>
<td>The type of antigen before processing by the immune system</td>
</tr>
<tr>
<td><a href="archival_id/">archival_id</a></td>
<td>Identifier for external archival resource for the investigation, e</td>
</tr>
<tr>
<td><a href="assay_type/">assay_type</a></td>
<td>The specific type of an assay</td>
</tr>
<tr>
<td><a href="assays/">assays</a></td>
<td>The assays performed by the investigation</td>
</tr>
<tr>
<td><a href="assessment_type/">assessment_type</a></td>
<td>The specific type of an assessment</td>
</tr>
<tr>
<td><a href="assessments/">assessments</a></td>
<td>The assessment that a dataset is about</td>
</tr>
<tr>
<td><a href="author/">author</a></td>
<td>Corresponding author</td>
</tr>
<tr>
<td><a href="authors/">authors</a></td>
<td>The authors of a reference</td>
</tr>
<tr>
<td><a href="biological_sex/">biological_sex</a></td>
<td>The biological sex of a participant</td>
</tr>
<tr>
<td><a href="biomaterial_provider/">biomaterial_provider</a></td>
<td>Name and address of the entity providing the sample</td>
</tr>
<tr>
<td><a href="c_alignment_end/">c_alignment_end</a></td>
<td>End position of the C gene alignment in both the sequence_alignment and germl...</td>
</tr>
<tr>
<td><a href="c_alignment_start/">c_alignment_start</a></td>
<td>Start position of the C gene alignment in both the sequence_alignment and ger...</td>
</tr>
<tr>
<td><a href="c_call/">c_call</a></td>
<td>Constant region gene with allele</td>
</tr>
<tr>
<td><a href="c_cigar/">c_cigar</a></td>
<td>CIGAR string for the C gene alignment</td>
</tr>
<tr>
<td><a href="c_germline_alignment/">c_germline_alignment</a></td>
<td>Aligned constant region germline sequence spanning the same region as the c_s...</td>
</tr>
<tr>
<td><a href="c_germline_alignment_aa/">c_germline_alignment_aa</a></td>
<td>Amino acid translation of the c_germline_aligment field</td>
</tr>
<tr>
<td><a href="c_germline_end/">c_germline_end</a></td>
<td>Alignment end position in the C gene reference sequence (1-based closed inter...</td>
</tr>
<tr>
<td><a href="c_germline_start/">c_germline_start</a></td>
<td>Alignment start position in the C gene reference sequence (1-based closed int...</td>
</tr>
<tr>
<td><a href="c_identity/">c_identity</a></td>
<td>Fractional identity for the C gene alignment</td>
</tr>
<tr>
<td><a href="c_score/">c_score</a></td>
<td>Alignment score for the C gene alignment</td>
</tr>
<tr>
<td><a href="c_sequence_alignment/">c_sequence_alignment</a></td>
<td>Aligned portion of query sequence assigned to the constant region, including ...</td>
</tr>
<tr>
<td><a href="c_sequence_alignment_aa/">c_sequence_alignment_aa</a></td>
<td>Amino acid translation of the c_sequence_alignment field</td>
</tr>
<tr>
<td><a href="c_sequence_end/">c_sequence_end</a></td>
<td>End position of the C gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="c_sequence_start/">c_sequence_start</a></td>
<td>Start position of the C gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="c_support/">c_support</a></td>
<td>C gene alignment E-value, p-value, likelihood, probability or other similar m...</td>
</tr>
<tr>
<td><a href="call/">call</a></td>
<td>Gene assignment with allele</td>
</tr>
<tr>
<td><a href="cdr1/">cdr1</a></td>
<td>Nucleotide sequence of the aligned CDR1 region</td>
</tr>
<tr>
<td><a href="cdr1_aa/">cdr1_aa</a></td>
<td>Amino acid translation of the cdr1 field</td>
</tr>
<tr>
<td><a href="cdr1_end/">cdr1_end</a></td>
<td>CDR1 end position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cdr1_start/">cdr1_start</a></td>
<td>CDR1 start position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cdr2/">cdr2</a></td>
<td>Nucleotide sequence of the aligned CDR2 region</td>
</tr>
<tr>
<td><a href="cdr2_aa/">cdr2_aa</a></td>
<td>Amino acid translation of the cdr2 field</td>
</tr>
<tr>
<td><a href="cdr2_end/">cdr2_end</a></td>
<td>CDR2 end position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cdr2_start/">cdr2_start</a></td>
<td>CDR2 start position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cdr3/">cdr3</a></td>
<td>Nucleotide sequence of the aligned CDR3 region</td>
</tr>
<tr>
<td><a href="cdr3_aa/">cdr3_aa</a></td>
<td>Amino acid translation of the cdr3 field</td>
</tr>
<tr>
<td><a href="cdr3_end/">cdr3_end</a></td>
<td>CDR3 end position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cdr3_start/">cdr3_start</a></td>
<td>CDR3 start position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="cell_id/">cell_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="cell_isolation/">cell_isolation</a></td>
<td>Description of the procedure used for marker-based isolation or enrich cells</td>
</tr>
<tr>
<td><a href="cell_number/">cell_number</a></td>
<td>Total number of cells that went into the experiment</td>
</tr>
<tr>
<td><a href="cell_phenotype/">cell_phenotype</a></td>
<td>List of cellular markers and their expression levels used to isolate the cell...</td>
</tr>
<tr>
<td><a href="cell_processing_protocol/">cell_processing_protocol</a></td>
<td>Description of the methods applied to the sample including cell preparation/ ...</td>
</tr>
<tr>
<td><a href="cell_quality/">cell_quality</a></td>
<td>Relative amount of viable cells after preparation and (if applicable) thawing</td>
</tr>
<tr>
<td><a href="cell_species/">cell_species</a></td>
<td>Binomial designation of the species from which the analyzed cells originate</td>
</tr>
<tr>
<td><a href="cell_storage/">cell_storage</a></td>
<td>TRUE if cells were cryo-preserved between isolation and further processing</td>
</tr>
<tr>
<td><a href="cell_subset/">cell_subset</a></td>
<td>Commonly-used designation of isolated cell population</td>
</tr>
<tr>
<td><a href="cells_per_reaction/">cells_per_reaction</a></td>
<td>Number of cells for each biological replicate</td>
</tr>
<tr>
<td><a href="chain_codomain/">chain_codomain</a></td>
<td>Immune receptor chain element in binary relation codomain</td>
</tr>
<tr>
<td><a href="chain_domain/">chain_domain</a></td>
<td>Immune receptor chain element in binary relation domain</td>
</tr>
<tr>
<td><a href="chain_similarity_type/">chain_similarity_type</a></td>
<td>Type of similarity calculation between two immune receptor chains</td>
</tr>
<tr>
<td><a href="chain_type/">chain_type</a></td>
<td>Gene locus (chain type)</td>
</tr>
<tr>
<td><a href="chains/">chains</a></td>
<td></td>
</tr>
<tr>
<td><a href="chromosome/">chromosome</a></td>
<td>chromosome on which the gene is located</td>
</tr>
<tr>
<td><a href="cigar/">cigar</a></td>
<td>Alignment CIGAR string</td>
</tr>
<tr>
<td><a href="clone_count/">clone_count</a></td>
<td>Absolute count of the size (number of members) of this clone in the repertoir...</td>
</tr>
<tr>
<td><a href="clone_id/">clone_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="coding_sequence/">coding_sequence</a></td>
<td>Nucleotide sequence of the core coding region, such as the coding region of a...</td>
</tr>
<tr>
<td><a href="collapsing_method/">collapsing_method</a></td>
<td>The method used for combining multiple sequences from (4) into a single seque...</td>
</tr>
<tr>
<td><a href="collected_by/">collected_by</a></td>
<td>Full contact information of the data collector, i</td>
</tr>
<tr>
<td><a href="collection_time_point_reference/">collection_time_point_reference</a></td>
<td>Event in the study schedule to which <code>Sample collection time</code> relates to</td>
</tr>
<tr>
<td><a href="collection_time_point_relative/">collection_time_point_relative</a></td>
<td>Time point at which sample was taken, relative to <code>Collection time event</code></td>
</tr>
<tr>
<td><a href="collection_time_point_relative_unit/">collection_time_point_relative_unit</a></td>
<td>Unit of Sample collection time</td>
</tr>
<tr>
<td><a href="complete_sequences/">complete_sequences</a></td>
<td>To be considered <code>complete</code>, the procedure used for library construction MUST...</td>
</tr>
<tr>
<td><a href="complete_vdj/">complete_vdj</a></td>
<td>True if the sequence alignment spans the entire V(D)J region</td>
</tr>
<tr>
<td><a href="conclusions/">conclusions</a></td>
<td>The conclusions from the investigation</td>
</tr>
<tr>
<td><a href="consensus_count/">consensus_count</a></td>
<td>Number of reads contributing to the UMI consensus or contig assembly for this...</td>
</tr>
<tr>
<td><a href="curation/">curation</a></td>
<td></td>
</tr>
<tr>
<td><a href="curational_tags/">curational_tags</a></td>
<td>Controlled-vocabulary tags applied to this description</td>
</tr>
<tr>
<td><a href="d2_alignment_end/">d2_alignment_end</a></td>
<td>End position of the second D gene alignment in both the sequence_alignment an...</td>
</tr>
<tr>
<td><a href="d2_alignment_start/">d2_alignment_start</a></td>
<td>Start position of the second D gene alignment in both the sequence_alignment ...</td>
</tr>
<tr>
<td><a href="d2_call/">d2_call</a></td>
<td>Second D gene with allele</td>
</tr>
<tr>
<td><a href="d2_cigar/">d2_cigar</a></td>
<td>CIGAR string for the second D gene alignment</td>
</tr>
<tr>
<td><a href="d2_frame/">d2_frame</a></td>
<td>Numerical reading frame (1, 2, 3) of the second D gene in the query nucleotid...</td>
</tr>
<tr>
<td><a href="d2_germline_alignment/">d2_germline_alignment</a></td>
<td>Aligned D gene germline sequence spanning the same region as the d2_sequence_...</td>
</tr>
<tr>
<td><a href="d2_germline_alignment_aa/">d2_germline_alignment_aa</a></td>
<td>Amino acid translation of the d2_germline_alignment field</td>
</tr>
<tr>
<td><a href="d2_germline_end/">d2_germline_end</a></td>
<td>Alignment end position in the second D gene reference sequence (1-based close...</td>
</tr>
<tr>
<td><a href="d2_germline_start/">d2_germline_start</a></td>
<td>Alignment start position in the second D gene reference sequence (1-based clo...</td>
</tr>
<tr>
<td><a href="d2_identity/">d2_identity</a></td>
<td>Fractional identity for the second D gene alignment</td>
</tr>
<tr>
<td><a href="d2_score/">d2_score</a></td>
<td>Alignment score for the second D gene alignment</td>
</tr>
<tr>
<td><a href="d2_sequence_alignment/">d2_sequence_alignment</a></td>
<td>Aligned portion of query sequence assigned to the second D gene, including an...</td>
</tr>
<tr>
<td><a href="d2_sequence_alignment_aa/">d2_sequence_alignment_aa</a></td>
<td>Amino acid translation of the d2_sequence_alignment field</td>
</tr>
<tr>
<td><a href="d2_sequence_end/">d2_sequence_end</a></td>
<td>End position of the second D gene in the query sequence (1-based closed inter...</td>
</tr>
<tr>
<td><a href="d2_sequence_start/">d2_sequence_start</a></td>
<td>Start position of the second D gene in the query sequence (1-based closed int...</td>
</tr>
<tr>
<td><a href="d2_support/">d2_support</a></td>
<td>D gene alignment E-value, p-value, likelihood, probability or other similar m...</td>
</tr>
<tr>
<td><a href="d_alignment_end/">d_alignment_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="d_alignment_start/">d_alignment_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="d_call/">d_call</a></td>
<td>First or only D gene with allele</td>
</tr>
<tr>
<td><a href="d_cigar/">d_cigar</a></td>
<td>CIGAR string for the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="d_frame/">d_frame</a></td>
<td>Numerical reading frame (1, 2, 3) of the first or only D gene in the query nu...</td>
</tr>
<tr>
<td><a href="d_germline_alignment/">d_germline_alignment</a></td>
<td>Aligned D gene germline sequence spanning the same region as the d_sequence_a...</td>
</tr>
<tr>
<td><a href="d_germline_alignment_aa/">d_germline_alignment_aa</a></td>
<td>Amino acid translation of the d_germline_alignment field</td>
</tr>
<tr>
<td><a href="d_germline_end/">d_germline_end</a></td>
<td>Alignment end position in the D gene reference sequence for the first or only...</td>
</tr>
<tr>
<td><a href="d_germline_start/">d_germline_start</a></td>
<td>Alignment start position in the D gene reference sequence for the first or on...</td>
</tr>
<tr>
<td><a href="d_identity/">d_identity</a></td>
<td>Fractional identity for the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="d_rs_3_prime_end/">d_rs_3_prime_end</a></td>
<td>End co-ordinate in the sequence field of the 3 prime D recombination site (D-...</td>
</tr>
<tr>
<td><a href="d_rs_3_prime_start/">d_rs_3_prime_start</a></td>
<td>Start co-ordinate in the sequence field of the 3 prime D recombination site (...</td>
</tr>
<tr>
<td><a href="d_rs_5_prime_end/">d_rs_5_prime_end</a></td>
<td>End co-ordinate in the sequence field of 5 the prime D recombination site (D-...</td>
</tr>
<tr>
<td><a href="d_rs_5_prime_start/">d_rs_5_prime_start</a></td>
<td>Start co-ordinate in the sequence field of the 5 prime D recombination site (...</td>
</tr>
<tr>
<td><a href="d_score/">d_score</a></td>
<td>Alignment score for the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="d_sequence_alignment/">d_sequence_alignment</a></td>
<td>Aligned portion of query sequence assigned to the first or only D gene, inclu...</td>
</tr>
<tr>
<td><a href="d_sequence_alignment_aa/">d_sequence_alignment_aa</a></td>
<td>Amino acid translation of the d_sequence_alignment field</td>
</tr>
<tr>
<td><a href="d_sequence_end/">d_sequence_end</a></td>
<td>End position of the first or only D gene in the query sequence</td>
</tr>
<tr>
<td><a href="d_sequence_start/">d_sequence_start</a></td>
<td>Start position of the first or only D gene in the query sequence</td>
</tr>
<tr>
<td><a href="d_support/">d_support</a></td>
<td>D gene alignment E-value, p-value, likelihood, probability or other similar m...</td>
</tr>
<tr>
<td><a href="data_location_type/">data_location_type</a></td>
<td>The type of location where data was found, e</td>
</tr>
<tr>
<td><a href="data_location_value/">data_location_value</a></td>
<td>An identifier for the location of the data, e</td>
</tr>
<tr>
<td><a href="data_processing/">data_processing</a></td>
<td>List of Data Processing objects</td>
</tr>
<tr>
<td><a href="data_processing_files/">data_processing_files</a></td>
<td>Array of file names for data produced by this data processing</td>
</tr>
<tr>
<td><a href="data_processing_id/">data_processing_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="data_processing_protocols/">data_processing_protocols</a></td>
<td>General description of how QC is performed</td>
</tr>
<tr>
<td><a href="datasets/">datasets</a></td>
<td>The datasets that support a conclusion</td>
</tr>
<tr>
<td><a href="deleted_genes/">deleted_genes</a></td>
<td>Array of genes identified as being deleted in this genotype</td>
</tr>
<tr>
<td><a href="delineation_scheme/">delineation_scheme</a></td>
<td>Name of the delineation scheme</td>
</tr>
<tr>
<td><a href="deposited_version/">deposited_version</a></td>
<td>Version number of the sequence within the repository</td>
</tr>
<tr>
<td><a href="derivation/">derivation</a></td>
<td>The class of nucleic acid that was used as primary starting material</td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>A human-readable description for a thing</td>
</tr>
<tr>
<td><a href="diagnosis/">diagnosis</a></td>
<td>Diagnosis information for subject</td>
</tr>
<tr>
<td><a href="disease/">disease</a></td>
<td>The disease relevant to an immune exposure</td>
</tr>
<tr>
<td><a href="disease_diagnosis/">disease_diagnosis</a></td>
<td>Diagnosis of subject</td>
</tr>
<tr>
<td><a href="disease_length/">disease_length</a></td>
<td>Time duration between initial diagnosis and current intervention</td>
</tr>
<tr>
<td><a href="disease_severity/">disease_severity</a></td>
<td>The severity of the disease relevant to an immune exposure</td>
</tr>
<tr>
<td><a href="disease_stage/">disease_stage</a></td>
<td>The stage of the disease relevant to an immune exposure</td>
</tr>
<tr>
<td><a href="disease_state_sample/">disease_state_sample</a></td>
<td>Histopathologic evaluation of the sample</td>
</tr>
<tr>
<td><a href="documented_alleles/">documented_alleles</a></td>
<td>List of alleles documented in reference set(s)</td>
</tr>
<tr>
<td><a href="documents/">documents</a></td>
<td>The documents produced by the investigation</td>
</tr>
<tr>
<td><a href="duplicate_count/">duplicate_count</a></td>
<td>Copy number or number of duplicate observations for the query sequence</td>
</tr>
<tr>
<td><a href="duration/">duration</a></td>
<td>The duration of this life event</td>
</tr>
<tr>
<td><a href="epitope/">epitope</a></td>
<td>The epitope being measured</td>
</tr>
<tr>
<td><a href="epitopes/">epitopes</a></td>
<td></td>
</tr>
<tr>
<td><a href="ethnicity/">ethnicity</a></td>
<td>The ethnicity of a participant</td>
</tr>
<tr>
<td><a href="exclusion_criteria/">exclusion_criteria</a></td>
<td>Participants are excluded from an investigation if they meet this criteria</td>
</tr>
<tr>
<td><a href="experiment_type/">experiment_type</a></td>
<td>The type of experiment that supports the conclusion</td>
</tr>
<tr>
<td><a href="exposure_material/">exposure_material</a></td>
<td>The material relevant to an immune exposure</td>
</tr>
<tr>
<td><a href="expression_id/">expression_id</a></td>
<td>Identifier of this expression property measurement</td>
</tr>
<tr>
<td><a href="expression_index/">expression_index</a></td>
<td>Index addressing the current event within the raw data set</td>
</tr>
<tr>
<td><a href="expression_raw_doi/">expression_raw_doi</a></td>
<td>DOI of raw data set containing the current event</td>
</tr>
<tr>
<td><a href="expression_study_method/">expression_study_method</a></td>
<td>Keyword describing the methodology used to assess expression</td>
</tr>
<tr>
<td><a href="file_type/">file_type</a></td>
<td>File format for the raw reads or sequences</td>
</tr>
<tr>
<td><a href="filename/">filename</a></td>
<td>File name for the raw reads or sequences</td>
</tr>
<tr>
<td><a href="forward_pcr_primer_target_location/">forward_pcr_primer_target_location</a></td>
<td>Position of the most distal nucleotide templated by the forward primer or pri...</td>
</tr>
<tr>
<td><a href="functional/">functional</a></td>
<td>True if the gene is functional, false if it is a pseudogene</td>
</tr>
<tr>
<td><a href="fwr1/">fwr1</a></td>
<td>Nucleotide sequence of the aligned FWR1 region</td>
</tr>
<tr>
<td><a href="fwr1_aa/">fwr1_aa</a></td>
<td>Amino acid translation of the fwr1 field</td>
</tr>
<tr>
<td><a href="fwr1_end/">fwr1_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr1_start/">fwr1_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr2/">fwr2</a></td>
<td>Nucleotide sequence of the aligned FWR2 region</td>
</tr>
<tr>
<td><a href="fwr2_aa/">fwr2_aa</a></td>
<td>Amino acid translation of the fwr2 field</td>
</tr>
<tr>
<td><a href="fwr2_end/">fwr2_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr2_start/">fwr2_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr3/">fwr3</a></td>
<td>Nucleotide sequence of the aligned FWR3 region</td>
</tr>
<tr>
<td><a href="fwr3_aa/">fwr3_aa</a></td>
<td>Amino acid translation of the fwr3 field</td>
</tr>
<tr>
<td><a href="fwr3_end/">fwr3_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr3_start/">fwr3_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="fwr4/">fwr4</a></td>
<td>Nucleotide sequence of the aligned FWR4 region</td>
</tr>
<tr>
<td><a href="fwr4_aa/">fwr4_aa</a></td>
<td>Amino acid translation of the fwr4 field</td>
</tr>
<tr>
<td><a href="fwr4_end/">fwr4_end</a></td>
<td>FWR4 end position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="fwr4_start/">fwr4_start</a></td>
<td>FWR4 start position in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="gene/">gene</a></td>
<td>The MHC gene to which the described allele belongs</td>
</tr>
<tr>
<td><a href="gene_designation/">gene_designation</a></td>
<td>Gene number or other identifier, as (and if) defined</td>
</tr>
<tr>
<td><a href="gene_end/">gene_end</a></td>
<td>Co-ordinate in the sequence field of the last gene-coding nucleotide in the c...</td>
</tr>
<tr>
<td><a href="gene_start/">gene_start</a></td>
<td>Co-ordinate in the sequence field of the first nucleotide in the coding_seque...</td>
</tr>
<tr>
<td><a href="genotype/">genotype</a></td>
<td></td>
</tr>
<tr>
<td><a href="genotype_class_list/">genotype_class_list</a></td>
<td>List of Genotypes included in this Receptor Genotype Set</td>
</tr>
<tr>
<td><a href="geolocation/">geolocation</a></td>
<td>The geolocation of a participant at birth</td>
</tr>
<tr>
<td><a href="germline_alignment/">germline_alignment</a></td>
<td></td>
</tr>
<tr>
<td><a href="germline_alignment_aa/">germline_alignment_aa</a></td>
<td></td>
</tr>
<tr>
<td><a href="germline_database/">germline_database</a></td>
<td>Source of germline V(D)J genes with version number or date accessed</td>
</tr>
<tr>
<td><a href="germline_end/">germline_end</a></td>
<td>Alignment end position in the reference sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="germline_set_id/">germline_set_id</a></td>
<td>Unique identifier of the GermlineSet within this file</td>
</tr>
<tr>
<td><a href="germline_set_name/">germline_set_name</a></td>
<td>descriptive name of this germline set</td>
</tr>
<tr>
<td><a href="germline_set_ref/">germline_set_ref</a></td>
<td></td>
</tr>
<tr>
<td><a href="germline_start/">germline_start</a></td>
<td>Alignment start position in the reference sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="gff_end/">gff_end</a></td>
<td>Genomic co-ordinates of the end of the sequence of interest described in this...</td>
</tr>
<tr>
<td><a href="gff_seqid/">gff_seqid</a></td>
<td>Sequence (from the assembly) of a window including the gene and preferably al...</td>
</tr>
<tr>
<td><a href="gff_start/">gff_start</a></td>
<td>Genomic co-ordinates of the start of the sequence of interest described in th...</td>
</tr>
<tr>
<td><a href="grants/">grants</a></td>
<td>Funding agencies and grant numbers</td>
</tr>
<tr>
<td><a href="identity/">identity</a></td>
<td>Alignment fractional identity</td>
</tr>
<tr>
<td><a href="IGH_chain/">IGH_chain</a></td>
<td>IG heavy chain</td>
</tr>
<tr>
<td><a href="IGK_chain/">IGK_chain</a></td>
<td>IG kappa light chain</td>
</tr>
<tr>
<td><a href="IGL_chain/">IGL_chain</a></td>
<td>IG lambda light chain</td>
</tr>
<tr>
<td><a href="immune_exposures/">immune_exposures</a></td>
<td></td>
</tr>
<tr>
<td><a href="immunogen/">immunogen</a></td>
<td>Antigen, vaccine or drug applied to subject at this intervention</td>
</tr>
<tr>
<td><a href="inclusion_criteria/">inclusion_criteria</a></td>
<td>Participants in an investigation must meet this criteria</td>
</tr>
<tr>
<td><a href="inclusion_exclusion_criteria/">inclusion_exclusion_criteria</a></td>
<td>List of criteria for inclusion/exclusion for the study</td>
</tr>
<tr>
<td><a href="index_filename/">index_filename</a></td>
<td>File name for the index file</td>
</tr>
<tr>
<td><a href="index_length/">index_length</a></td>
<td>Read length in bases for the index file</td>
</tr>
<tr>
<td><a href="inference_process/">inference_process</a></td>
<td>Information on how the genotype was acquired</td>
</tr>
<tr>
<td><a href="inference_type/">inference_type</a></td>
<td>Type of inference(s) from which this gene sequence was inferred</td>
</tr>
<tr>
<td><a href="institution_name/">institution_name</a></td>
<td>Individual's department and institution name</td>
</tr>
<tr>
<td><a href="intervention/">intervention</a></td>
<td>Description of intervention</td>
</tr>
<tr>
<td><a href="investigation/">investigation</a></td>
<td>An investigation in which the study arm participates</td>
</tr>
<tr>
<td><a href="investigations/">investigations</a></td>
<td>The investigations that a reference or conclusion are about</td>
</tr>
<tr>
<td><a href="isotype/">isotype</a></td>
<td>The isotype of the chain</td>
</tr>
<tr>
<td><a href="issue/">issue</a></td>
<td>The issue of the journal in which a reference was published</td>
</tr>
<tr>
<td><a href="j_alignment_end/">j_alignment_end</a></td>
<td>End position of the J gene alignment in both the sequence_alignment and germl...</td>
</tr>
<tr>
<td><a href="j_alignment_start/">j_alignment_start</a></td>
<td>Start position of the J gene alignment in both the sequence_alignment and ger...</td>
</tr>
<tr>
<td><a href="j_call/">j_call</a></td>
<td>J gene with allele</td>
</tr>
<tr>
<td><a href="j_cdr3_end/">j_cdr3_end</a></td>
<td>In the case of a J-gene, the co-ordinate in the sequence field of the first n...</td>
</tr>
<tr>
<td><a href="j_cigar/">j_cigar</a></td>
<td>CIGAR string for the J gene alignment</td>
</tr>
<tr>
<td><a href="j_codon_frame/">j_codon_frame</a></td>
<td>Codon position of the first nucleotide in the 'coding_sequence' field</td>
</tr>
<tr>
<td><a href="j_donor_splice/">j_donor_splice</a></td>
<td>Co-ordinate in the sequence field of the final 3' nucleotide of the J-REGION ...</td>
</tr>
<tr>
<td><a href="j_frameshift/">j_frameshift</a></td>
<td>True if the J gene in the query nucleotide sequence contains a translational ...</td>
</tr>
<tr>
<td><a href="j_germline_alignment/">j_germline_alignment</a></td>
<td>Aligned J gene germline sequence spanning the same region as the j_sequence_a...</td>
</tr>
<tr>
<td><a href="j_germline_alignment_aa/">j_germline_alignment_aa</a></td>
<td>Amino acid translation of the j_germline_alignment field</td>
</tr>
<tr>
<td><a href="j_germline_end/">j_germline_end</a></td>
<td>Alignment end position in the J gene reference sequence (1-based closed inter...</td>
</tr>
<tr>
<td><a href="j_germline_start/">j_germline_start</a></td>
<td>Alignment start position in the J gene reference sequence (1-based closed int...</td>
</tr>
<tr>
<td><a href="j_identity/">j_identity</a></td>
<td>Fractional identity for the J gene alignment</td>
</tr>
<tr>
<td><a href="j_rs_end/">j_rs_end</a></td>
<td>End co-ordinate in the sequence field of J recombination site (J-genes only)</td>
</tr>
<tr>
<td><a href="j_rs_start/">j_rs_start</a></td>
<td>Start co-ordinate in the sequence field of J recombination site (J-genes only...</td>
</tr>
<tr>
<td><a href="j_score/">j_score</a></td>
<td>Alignment score for the J gene alignment</td>
</tr>
<tr>
<td><a href="j_sequence_alignment/">j_sequence_alignment</a></td>
<td>Aligned portion of query sequence assigned to the J gene, including any indel...</td>
</tr>
<tr>
<td><a href="j_sequence_alignment_aa/">j_sequence_alignment_aa</a></td>
<td>Amino acid translation of the j_sequence_alignment field</td>
</tr>
<tr>
<td><a href="j_sequence_end/">j_sequence_end</a></td>
<td>End position of the J gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="j_sequence_start/">j_sequence_start</a></td>
<td>Start position of the J gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="j_support/">j_support</a></td>
<td>J gene alignment E-value, p-value, likelihood, probability or other similar m...</td>
</tr>
<tr>
<td><a href="journal/">journal</a></td>
<td>The journal in which a reference was published</td>
</tr>
<tr>
<td><a href="junction/">junction</a></td>
<td></td>
</tr>
<tr>
<td><a href="junction_aa/">junction_aa</a></td>
<td>Amino acid form of the junction</td>
</tr>
<tr>
<td><a href="junction_aa_length/">junction_aa_length</a></td>
<td></td>
</tr>
<tr>
<td><a href="junction_end/">junction_end</a></td>
<td>Junction region end position in the alignment (1-based closed interval)</td>
</tr>
<tr>
<td><a href="junction_length/">junction_length</a></td>
<td></td>
</tr>
<tr>
<td><a href="junction_start/">junction_start</a></td>
<td>Junction region start position in the alignment (1-based closed interval)</td>
</tr>
<tr>
<td><a href="keywords_study/">keywords_study</a></td>
<td>Keywords describing properties of one or more data sets in a study</td>
</tr>
<tr>
<td><a href="lab_address/">lab_address</a></td>
<td></td>
</tr>
<tr>
<td><a href="lab_name/">lab_name</a></td>
<td></td>
</tr>
<tr>
<td><a href="label/">label</a></td>
<td></td>
</tr>
<tr>
<td><a href="leader_1_end/">leader_1_end</a></td>
<td>End co-ordinate in the sequence field of L-PART1 (V-genes only)</td>
</tr>
<tr>
<td><a href="leader_1_start/">leader_1_start</a></td>
<td>Start co-ordinate in the sequence field of L-PART1 (V-genes only)</td>
</tr>
<tr>
<td><a href="leader_2_end/">leader_2_end</a></td>
<td>End co-ordinate in the sequence field of L-PART2 (V-genes only)</td>
</tr>
<tr>
<td><a href="leader_2_start/">leader_2_start</a></td>
<td>Start co-ordinate in the sequence field of L-PART2 (V-genes only)</td>
</tr>
<tr>
<td><a href="library_generation_kit_version/">library_generation_kit_version</a></td>
<td>When using a library generation protocol from a commercial provider, provide ...</td>
</tr>
<tr>
<td><a href="library_generation_method/">library_generation_method</a></td>
<td>Generic type of library generation</td>
</tr>
<tr>
<td><a href="library_generation_protocol/">library_generation_protocol</a></td>
<td>Description of processes applied to substrate to obtain a library that is rea...</td>
</tr>
<tr>
<td><a href="life_event/">life_event</a></td>
<td>The life event corresponding to an immune exposure</td>
</tr>
<tr>
<td><a href="life_event_type/">life_event_type</a></td>
<td>The specific type of a life event</td>
</tr>
<tr>
<td><a href="life_events/">life_events</a></td>
<td></td>
</tr>
<tr>
<td><a href="ligand_type/">ligand_type</a></td>
<td>Classification of ligand binding to receptor</td>
</tr>
<tr>
<td><a href="link_type/">link_type</a></td>
<td>Relation between subject and <code>linked_subjects</code>, can be genetic or environment...</td>
</tr>
<tr>
<td><a href="linked_subjects/">linked_subjects</a></td>
<td>Subject ID to which <code>Relation type</code> refers</td>
</tr>
<tr>
<td><a href="locus/">locus</a></td>
<td></td>
</tr>
<tr>
<td><a href="maintainer/">maintainer</a></td>
<td>Maintainer of this sequence record</td>
</tr>
<tr>
<td><a href="medical_history/">medical_history</a></td>
<td>Medical history of subject that is relevant to assess the course of disease a...</td>
</tr>
<tr>
<td><a href="mhc_allele_1/">mhc_allele_1</a></td>
<td>Allele designation of the MHC alpha chain</td>
</tr>
<tr>
<td><a href="mhc_allele_2/">mhc_allele_2</a></td>
<td>Allele designation of the MHC class II beta chain or the invariant beta2-micr...</td>
</tr>
<tr>
<td><a href="mhc_alleles/">mhc_alleles</a></td>
<td>List of MHC alleles of the indicated mhc_class identified in an individual</td>
</tr>
<tr>
<td><a href="mhc_class/">mhc_class</a></td>
<td></td>
</tr>
<tr>
<td><a href="mhc_gene_1/">mhc_gene_1</a></td>
<td>The MHC gene to which the mhc_allele_1 belongs</td>
</tr>
<tr>
<td><a href="mhc_gene_2/">mhc_gene_2</a></td>
<td>The MHC gene to which the mhc_allele_2 belongs</td>
</tr>
<tr>
<td><a href="mhc_genotype_id/">mhc_genotype_id</a></td>
<td>A unique identifier for this MHCGenotype, assumed to be unique in the context...</td>
</tr>
<tr>
<td><a href="mhc_genotype_list/">mhc_genotype_list</a></td>
<td>List of MHCGenotypes included in this set</td>
</tr>
<tr>
<td><a href="mhc_genotype_set/">mhc_genotype_set</a></td>
<td>MHC genotype set for this subject</td>
</tr>
<tr>
<td><a href="mhc_genotype_set_id/">mhc_genotype_set_id</a></td>
<td>A unique identifier for this MHCGenotypeSet</td>
</tr>
<tr>
<td><a href="mhc_genotyping_method/">mhc_genotyping_method</a></td>
<td>Information on how the genotype was determined</td>
</tr>
<tr>
<td><a href="month/">month</a></td>
<td>The month of the issue of the journal in which a reference was published</td>
</tr>
<tr>
<td><a href="n1_length/">n1_length</a></td>
<td>Number of untemplated nucleotides 5' of the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="n2_length/">n2_length</a></td>
<td>Number of untemplated nucleotides 3' of the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="n3_length/">n3_length</a></td>
<td>Number of untemplated nucleotides 3' of the second D gene alignment</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>A human-readable name for a thing</td>
</tr>
<tr>
<td><a href="newick/">newick</a></td>
<td>Newick string of the tree edges</td>
</tr>
<tr>
<td><a href="nodes/">nodes</a></td>
<td>Dictionary of nodes in the tree, keyed by sequence_id string</td>
</tr>
<tr>
<td><a href="np1/">np1</a></td>
<td>Nucleotide sequence of the combined N/P region between the V gene and first D...</td>
</tr>
<tr>
<td><a href="np1_aa/">np1_aa</a></td>
<td>Amino acid translation of the np1 field</td>
</tr>
<tr>
<td><a href="np1_length/">np1_length</a></td>
<td>Number of nucleotides between the V gene and first D gene alignments or betwe...</td>
</tr>
<tr>
<td><a href="np2/">np2</a></td>
<td>Nucleotide sequence of the combined N/P region between either the first D gen...</td>
</tr>
<tr>
<td><a href="np2_aa/">np2_aa</a></td>
<td>Amino acid translation of the np2 field</td>
</tr>
<tr>
<td><a href="np2_length/">np2_length</a></td>
<td>Number of nucleotides between either the first D gene and J gene alignments o...</td>
</tr>
<tr>
<td><a href="np3/">np3</a></td>
<td>Nucleotide sequence of the combined N/P region between the second D gene and ...</td>
</tr>
<tr>
<td><a href="np3_aa/">np3_aa</a></td>
<td>Amino acid translation of the np3 field</td>
</tr>
<tr>
<td><a href="np3_length/">np3_length</a></td>
<td>Number of nucleotides between the second D gene and J gene alignments</td>
</tr>
<tr>
<td><a href="observation_type/">observation_type</a></td>
<td>The type of observation from which this sequence was drawn, such as direct se...</td>
</tr>
<tr>
<td><a href="orcid_id/">orcid_id</a></td>
<td>Individual's ORCID identifier</td>
</tr>
<tr>
<td><a href="organism/">organism</a></td>
<td>The type of organism that the conclusion is about</td>
</tr>
<tr>
<td><a href="p3d2_length/">p3d2_length</a></td>
<td>Number of palindromic nucleotides 3' of the second D gene alignment</td>
</tr>
<tr>
<td><a href="p3d_length/">p3d_length</a></td>
<td>Number of palindromic nucleotides 3' of the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="p3v_length/">p3v_length</a></td>
<td>Number of palindromic nucleotides 3' of the V gene alignment</td>
</tr>
<tr>
<td><a href="p5d2_length/">p5d2_length</a></td>
<td>Number of palindromic nucleotides 5' of the second D gene alignment</td>
</tr>
<tr>
<td><a href="p5d_length/">p5d_length</a></td>
<td>Number of palindromic nucleotides 5' of the first or only D gene alignment</td>
</tr>
<tr>
<td><a href="p5j_length/">p5j_length</a></td>
<td>Number of palindromic nucleotides 5' of the J gene alignment</td>
</tr>
<tr>
<td><a href="pages/">pages</a></td>
<td>The pages of the issue of the journal in which a reference was published</td>
</tr>
<tr>
<td><a href="paired_filename/">paired_filename</a></td>
<td>File name for the second file in paired-read sequencing</td>
</tr>
<tr>
<td><a href="paired_read_direction/">paired_read_direction</a></td>
<td>Read direction for the second file in paired-read sequencing</td>
</tr>
<tr>
<td><a href="paired_read_length/">paired_read_length</a></td>
<td>Read length in bases for the second file in paired-read sequencing</td>
</tr>
<tr>
<td><a href="paired_reads_assembly/">paired_reads_assembly</a></td>
<td>How paired end reads were assembled into a single receptor sequence</td>
</tr>
<tr>
<td><a href="paralogs/">paralogs</a></td>
<td>Gene symbols of any paralogs</td>
</tr>
<tr>
<td><a href="participant/">participant</a></td>
<td>The participant of a life event</td>
</tr>
<tr>
<td><a href="participants/">participants</a></td>
<td>The participants involved with the investigation</td>
</tr>
<tr>
<td><a href="patch_no/">patch_no</a></td>
<td>Genome assembly patch number in which this gene was determined</td>
</tr>
<tr>
<td><a href="pcr_target/">pcr_target</a></td>
<td>Target locus for PCR</td>
</tr>
<tr>
<td><a href="pcr_target_locus/">pcr_target_locus</a></td>
<td>Designation of the target locus</td>
</tr>
<tr>
<td><a href="peptide_end/">peptide_end</a></td>
<td>End position of the peptide within the reference protein sequence</td>
</tr>
<tr>
<td><a href="peptide_start/">peptide_start</a></td>
<td>Start position of the peptide within the reference protein sequence</td>
</tr>
<tr>
<td><a href="phasing/">phasing</a></td>
<td>Chromosomal phasing indicator</td>
</tr>
<tr>
<td><a href="phenotypic_sex/">phenotypic_sex</a></td>
<td>The phenotypic sex of a participant</td>
</tr>
<tr>
<td><a href="physical_linkage/">physical_linkage</a></td>
<td>In case an experimental setup is used that physically links nucleic acids der...</td>
</tr>
<tr>
<td><a href="primary_annotation/">primary_annotation</a></td>
<td>If true, indicates this is the primary or default data processing for the rep...</td>
</tr>
<tr>
<td><a href="primer_match_cutoffs/">primer_match_cutoffs</a></td>
<td>How primers were identified in the sequences, were they removed/masked/etc?</td>
</tr>
<tr>
<td><a href="prior_therapies/">prior_therapies</a></td>
<td>List of all relevant previous therapies applied to subject for treatment of `...</td>
</tr>
<tr>
<td><a href="process/">process</a></td>
<td>The type of specimen collection process that resulted in this specimen</td>
</tr>
<tr>
<td><a href="productive/">productive</a></td>
<td>True if the V(D)J sequence is predicted to be productive</td>
</tr>
<tr>
<td><a href="property/">property</a></td>
<td>Name of the property observed, typically a gene or antibody identifier (and l...</td>
</tr>
<tr>
<td><a href="property_type/">property_type</a></td>
<td>Keyword describing the property type and detection method used to measure the...</td>
</tr>
<tr>
<td><a href="pub_ids/">pub_ids</a></td>
<td></td>
</tr>
<tr>
<td><a href="quality/">quality</a></td>
<td>The Sanger/Phred quality scores for assessment of sequence quality</td>
</tr>
<tr>
<td><a href="quality_alignment/">quality_alignment</a></td>
<td>Sanger/Phred quality scores for assessment of sequence_alignment quality</td>
</tr>
<tr>
<td><a href="quality_thresholds/">quality_thresholds</a></td>
<td>How/if sequences were removed from (4) based on base quality scores</td>
</tr>
<tr>
<td><a href="race/">race</a></td>
<td>The race of a participant</td>
</tr>
<tr>
<td><a href="rank/">rank</a></td>
<td>Alignment rank</td>
</tr>
<tr>
<td><a href="reactivity_measurements/">reactivity_measurements</a></td>
<td>Records of reactivity measurement</td>
</tr>
<tr>
<td><a href="reactivity_method/">reactivity_method</a></td>
<td>The methodology used to assess expression (assay implemented in experiment)</td>
</tr>
<tr>
<td><a href="reactivity_readout/">reactivity_readout</a></td>
<td>Reactivity measurement read-out</td>
</tr>
<tr>
<td><a href="reactivity_unit/">reactivity_unit</a></td>
<td>The unit of the measurement</td>
</tr>
<tr>
<td><a href="reactivity_value/">reactivity_value</a></td>
<td>The absolute (processed) value of the measurement</td>
</tr>
<tr>
<td><a href="read_direction/">read_direction</a></td>
<td>Read direction for the raw reads or sequences</td>
</tr>
<tr>
<td><a href="read_length/">read_length</a></td>
<td>Read length in bases for the first file in paired-read sequencing</td>
</tr>
<tr>
<td><a href="rearranged_support/">rearranged_support</a></td>
<td></td>
</tr>
<tr>
<td><a href="rearrangements/">rearrangements</a></td>
<td>Array of sequence identifiers defined for the Rearrangement object</td>
</tr>
<tr>
<td><a href="receptor_genotype_id/">receptor_genotype_id</a></td>
<td>A unique identifier within the file for this Receptor Genotype, typically gen...</td>
</tr>
<tr>
<td><a href="receptor_genotype_set/">receptor_genotype_set</a></td>
<td>Immune receptor genotype set for this subject</td>
</tr>
<tr>
<td><a href="receptor_genotype_set_id/">receptor_genotype_set_id</a></td>
<td>A unique identifier for this Receptor Genotype Set, typically generated by th...</td>
</tr>
<tr>
<td><a href="receptor_hash/">receptor_hash</a></td>
<td>The SHA256 hash of the receptor amino acid sequence, calculated on the concat...</td>
</tr>
<tr>
<td><a href="receptor_id/">receptor_id</a></td>
<td>ID of the current Receptor object, unique within the local repository</td>
</tr>
<tr>
<td><a href="receptor_ref/">receptor_ref</a></td>
<td>Array of receptor identifiers defined for the Receptor object</td>
</tr>
<tr>
<td><a href="receptor_type/">receptor_type</a></td>
<td>The top-level receptor type, either Immunoglobulin (Ig) or T Cell Receptor (T...</td>
</tr>
<tr>
<td><a href="receptor_variable_domain_1_aa/">receptor_variable_domain_1_aa</a></td>
<td>Complete amino acid sequence of the mature variable domain of the Ig heavy, T...</td>
</tr>
<tr>
<td><a href="receptor_variable_domain_1_locus/">receptor_variable_domain_1_locus</a></td>
<td>Locus from which the variable domain in receptor_variable_domain_1_aa origina...</td>
</tr>
<tr>
<td><a href="receptor_variable_domain_2_aa/">receptor_variable_domain_2_aa</a></td>
<td>Complete amino acid sequence of the mature variable domain of the Ig light, T...</td>
</tr>
<tr>
<td><a href="receptor_variable_domain_2_locus/">receptor_variable_domain_2_locus</a></td>
<td>Locus from which the variable domain in receptor_variable_domain_2_aa origina...</td>
</tr>
<tr>
<td><a href="receptors/">receptors</a></td>
<td>Array of receptor identifiers defined for the Receptor object</td>
</tr>
<tr>
<td><a href="reference_set_ref/">reference_set_ref</a></td>
<td>Repository and list from which it was taken (issuer/name/version)</td>
</tr>
<tr>
<td><a href="references/">references</a></td>
<td></td>
</tr>
<tr>
<td><a href="release_date/">release_date</a></td>
<td>Initial release of the investigation and its data</td>
</tr>
<tr>
<td><a href="release_description/">release_description</a></td>
<td>Brief descriptive notes of the reason for this release and the changes embodi...</td>
</tr>
<tr>
<td><a href="release_version/">release_version</a></td>
<td></td>
</tr>
<tr>
<td><a href="repertoire_description/">repertoire_description</a></td>
<td></td>
</tr>
<tr>
<td><a href="repertoire_group_description/">repertoire_group_description</a></td>
<td>Repertoire collection description</td>
</tr>
<tr>
<td><a href="repertoire_group_id/">repertoire_group_id</a></td>
<td>Identifier for this repertoire collection</td>
</tr>
<tr>
<td><a href="repertoire_group_name/">repertoire_group_name</a></td>
<td>Short display name for this repertoire collection</td>
</tr>
<tr>
<td><a href="repertoire_id/">repertoire_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="repertoire_name/">repertoire_name</a></td>
<td>Short generic display name for the repertoire</td>
</tr>
<tr>
<td><a href="repertoires/">repertoires</a></td>
<td>List of repertoires in this collection with an associated description and tim...</td>
</tr>
<tr>
<td><a href="repository_name/">repository_name</a></td>
<td></td>
</tr>
<tr>
<td><a href="repository_ref/">repository_ref</a></td>
<td>Queryable id or accession number of the sequence published by the repository</td>
</tr>
<tr>
<td><a href="result/">result</a></td>
<td>The content of the conclusion</td>
</tr>
<tr>
<td><a href="rev_comp/">rev_comp</a></td>
<td></td>
</tr>
<tr>
<td><a href="reverse_pcr_primer_target_location/">reverse_pcr_primer_target_location</a></td>
<td>Position of the most proximal nucleotide templated by the reverse primer or p...</td>
</tr>
<tr>
<td><a href="sample/">sample</a></td>
<td>List of Sample Processing objects</td>
</tr>
<tr>
<td><a href="sample_id/">sample_id</a></td>
<td>Sample ID assigned by submitter, unique within study</td>
</tr>
<tr>
<td><a href="sample_processing_id/">sample_processing_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="sample_type/">sample_type</a></td>
<td>The way the sample was obtained, e</td>
</tr>
<tr>
<td><a href="score/">score</a></td>
<td>Alignment score</td>
</tr>
<tr>
<td><a href="seed_id/">seed_id</a></td>
<td>sequence_id of the seed sequence</td>
</tr>
<tr>
<td><a href="segment/">segment</a></td>
<td>The segment for this alignment</td>
</tr>
<tr>
<td><a href="sequence/">sequence</a></td>
<td>Nucleotide sequence</td>
</tr>
<tr>
<td><a href="sequence_aa/">sequence_aa</a></td>
<td>Amino acid sequence</td>
</tr>
<tr>
<td><a href="sequence_alignment/">sequence_alignment</a></td>
<td></td>
</tr>
<tr>
<td><a href="sequence_alignment_aa/">sequence_alignment_aa</a></td>
<td>Amino acid translation of the aligned query sequence</td>
</tr>
<tr>
<td><a href="sequence_delineation_id/">sequence_delineation_id</a></td>
<td>Unique identifier of this SequenceDelineationV within the file</td>
</tr>
<tr>
<td><a href="sequence_end/">sequence_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="sequence_id/">sequence_id</a></td>
<td></td>
</tr>
<tr>
<td><a href="sequence_start/">sequence_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="sequence_type/">sequence_type</a></td>
<td>Sequence type (V, D, J, C)</td>
</tr>
<tr>
<td><a href="sequences/">sequences</a></td>
<td>List sequence_id strings that act as keys to the Rearrangement records for me...</td>
</tr>
<tr>
<td><a href="sequencing_data_id/">sequencing_data_id</a></td>
<td>Persistent identifier of raw data stored in an archive (e</td>
</tr>
<tr>
<td><a href="sequencing_facility/">sequencing_facility</a></td>
<td>Name and address of sequencing facility</td>
</tr>
<tr>
<td><a href="sequencing_files/">sequencing_files</a></td>
<td>Set of sequencing files produced by the sequencing run</td>
</tr>
<tr>
<td><a href="sequencing_kit/">sequencing_kit</a></td>
<td>Name, manufacturer, order and lot numbers of sequencing kit</td>
</tr>
<tr>
<td><a href="sequencing_platform/">sequencing_platform</a></td>
<td>Designation of sequencing instrument used</td>
</tr>
<tr>
<td><a href="sequencing_run_date/">sequencing_run_date</a></td>
<td>Date of sequencing run</td>
</tr>
<tr>
<td><a href="sequencing_run_id/">sequencing_run_id</a></td>
<td>ID of sequencing run assigned by the sequencing facility</td>
</tr>
<tr>
<td><a href="sex/">sex</a></td>
<td>Biological sex of subject</td>
</tr>
<tr>
<td><a href="simulations/">simulations</a></td>
<td>The simulations performed by the investigation</td>
</tr>
<tr>
<td><a href="single_cell/">single_cell</a></td>
<td>TRUE if single cells were isolated into separate compartments</td>
</tr>
<tr>
<td><a href="software_versions/">software_versions</a></td>
<td>Version number and / or date, include company pipelines</td>
</tr>
<tr>
<td><a href="source_organism/">source_organism</a></td>
<td>The organism that the source protein comes from</td>
</tr>
<tr>
<td><a href="source_protein/">source_protein</a></td>
<td>The protein that this epitope comes from</td>
</tr>
<tr>
<td><a href="source_uri/">source_uri</a></td>
<td>AKC reference to a foreign thing</td>
</tr>
<tr>
<td><a href="sources/">sources</a></td>
<td>The source URLs for a reference</td>
</tr>
<tr>
<td><a href="species/">species</a></td>
<td>The species of a participant</td>
</tr>
<tr>
<td><a href="species_subgroup/">species_subgroup</a></td>
<td>Race, strain or other species subgroup to which this subject belongs</td>
</tr>
<tr>
<td><a href="species_subgroup_type/">species_subgroup_type</a></td>
<td></td>
</tr>
<tr>
<td><a href="specimen/">specimen</a></td>
<td>The specimen that was input for an assay</td>
</tr>
<tr>
<td><a href="specimen_collections/">specimen_collections</a></td>
<td></td>
</tr>
<tr>
<td><a href="specimen_processing/">specimen_processing</a></td>
<td>A series of zero or more specimen processing steps that precede an assay</td>
</tr>
<tr>
<td><a href="specimen_type/">specimen_type</a></td>
<td>The type of this specimen</td>
</tr>
<tr>
<td><a href="specimens/">specimens</a></td>
<td></td>
</tr>
<tr>
<td><a href="start/">start</a></td>
<td>The start time of this life event, relative to the T0 event</td>
</tr>
<tr>
<td><a href="status/">status</a></td>
<td>Status of record, assumed active if the field is not present</td>
</tr>
<tr>
<td><a href="stop_codon/">stop_codon</a></td>
<td>True if the aligned sequence contains a stop codon</td>
</tr>
<tr>
<td><a href="strain/">strain</a></td>
<td>The strain of the participant (non-human study participants)</td>
</tr>
<tr>
<td><a href="strain_name/">strain_name</a></td>
<td>Non-human designation of the strain or breed of animal used</td>
</tr>
<tr>
<td><a href="strand/">strand</a></td>
<td>sense (+ or -)</td>
</tr>
<tr>
<td><a href="study/">study</a></td>
<td>Study object</td>
</tr>
<tr>
<td><a href="study_arm/">study_arm</a></td>
<td>The study arm that a participant is a member of</td>
</tr>
<tr>
<td><a href="study_arms/">study_arms</a></td>
<td>The study arms that are relevant for a study event</td>
</tr>
<tr>
<td><a href="study_contact/">study_contact</a></td>
<td>Full contact information of the contact persons for this study This should in...</td>
</tr>
<tr>
<td><a href="study_description/">study_description</a></td>
<td>Generic study description</td>
</tr>
<tr>
<td><a href="study_event/">study_event</a></td>
<td>The study event corresponding to a life event</td>
</tr>
<tr>
<td><a href="study_events/">study_events</a></td>
<td></td>
</tr>
<tr>
<td><a href="study_group_description/">study_group_description</a></td>
<td>Designation of study arm to which the subject is assigned to</td>
</tr>
<tr>
<td><a href="study_id/">study_id</a></td>
<td>Unique ID assigned by study registry such as one of the International Nucleot...</td>
</tr>
<tr>
<td><a href="study_title/">study_title</a></td>
<td>Descriptive study title</td>
</tr>
<tr>
<td><a href="study_type/">study_type</a></td>
<td>Type of study design</td>
</tr>
<tr>
<td><a href="subgroup_designation/">subgroup_designation</a></td>
<td>Identifier of the gene subgroup or clade, as (and if) defined</td>
</tr>
<tr>
<td><a href="subject/">subject</a></td>
<td>Subject object</td>
</tr>
<tr>
<td><a href="subject_id/">subject_id</a></td>
<td>Subject ID assigned by submitter, unique within study</td>
</tr>
<tr>
<td><a href="submitted_by/">submitted_by</a></td>
<td>Full contact information of the data depositor, i</td>
</tr>
<tr>
<td><a href="support/">support</a></td>
<td>Alignment E-value, p-value, likelihood, probability or other similar measure ...</td>
</tr>
<tr>
<td><a href="synthetic/">synthetic</a></td>
<td>TRUE for libraries in which the diversity has been synthetically generated (e</td>
</tr>
<tr>
<td><a href="t0_event/">t0_event</a></td>
<td>The T0 event used to specify the time of this life event</td>
</tr>
<tr>
<td><a href="t0_event_type/">t0_event_type</a></td>
<td>The type of the T0 event used to specify the time of this life event</td>
</tr>
<tr>
<td><a href="target_entity_type/">target_entity_type</a></td>
<td>The type of the entity being measured</td>
</tr>
<tr>
<td><a href="tcell_receptors/">tcell_receptors</a></td>
<td>The T cell receptors being measured</td>
</tr>
<tr>
<td><a href="template_amount/">template_amount</a></td>
<td>Amount of template that went into the process</td>
</tr>
<tr>
<td><a href="template_amount_unit/">template_amount_unit</a></td>
<td>Unit of template amount</td>
</tr>
<tr>
<td><a href="template_class/">template_class</a></td>
<td>The class of nucleic acid that was used as primary starting material for the ...</td>
</tr>
<tr>
<td><a href="template_quality/">template_quality</a></td>
<td>Description and results of the quality control performed on the template mate...</td>
</tr>
<tr>
<td><a href="time_point/">time_point</a></td>
<td>Time point designation for this repertoire within the group</td>
</tr>
<tr>
<td><a href="time_unit/">time_unit</a></td>
<td>The time unit used to measure the start and duration of this life event</td>
</tr>
<tr>
<td><a href="tissue/">tissue</a></td>
<td>The tissue from which this specimen was taken</td>
</tr>
<tr>
<td><a href="tissue_processing/">tissue_processing</a></td>
<td>Enzymatic digestion and/or physical methods used to isolate cells from sample</td>
</tr>
<tr>
<td><a href="title/">title</a></td>
<td>The title of a reference</td>
</tr>
<tr>
<td><a href="total_reads_passing_qc_filter/">total_reads_passing_qc_filter</a></td>
<td>Number of usable reads for analysis</td>
</tr>
<tr>
<td><a href="TRA_chain/">TRA_chain</a></td>
<td>T cell receptor alpha chain</td>
</tr>
<tr>
<td><a href="TRB_chain/">TRB_chain</a></td>
<td>T cell receptor beta chain</td>
</tr>
<tr>
<td><a href="TRD_chain/">TRD_chain</a></td>
<td>T cell receptor delta chain</td>
</tr>
<tr>
<td><a href="tree_id/">tree_id</a></td>
<td>Identifier for the tree</td>
</tr>
<tr>
<td><a href="TRG_chain/">TRG_chain</a></td>
<td>T cell receptor gamma chain</td>
</tr>
<tr>
<td><a href="type/">type</a></td>
<td></td>
</tr>
<tr>
<td><a href="umi_count/">umi_count</a></td>
<td></td>
</tr>
<tr>
<td><a href="unaligned_sequence/">unaligned_sequence</a></td>
<td>entire V-sequence covered by this delineation</td>
</tr>
<tr>
<td><a href="undocumented_alleles/">undocumented_alleles</a></td>
<td>List of alleles inferred to be present and not documented in an identified Ge...</td>
</tr>
<tr>
<td><a href="unit/">unit</a></td>
<td>The measurement result value unit</td>
</tr>
<tr>
<td><a href="unrearranged_support/">unrearranged_support</a></td>
<td></td>
</tr>
<tr>
<td><a href="update_date/">update_date</a></td>
<td>Subsequence updates to the investigation or its data</td>
</tr>
<tr>
<td><a href="utr_5_prime_end/">utr_5_prime_end</a></td>
<td>End co-ordinate in the sequence field of the 5 prime UTR (V-genes only)</td>
</tr>
<tr>
<td><a href="utr_5_prime_start/">utr_5_prime_start</a></td>
<td>Start co-ordinate in the sequence field of the 5 prime UTR (V-genes only)</td>
</tr>
<tr>
<td><a href="v_alignment_end/">v_alignment_end</a></td>
<td></td>
</tr>
<tr>
<td><a href="v_alignment_start/">v_alignment_start</a></td>
<td></td>
</tr>
<tr>
<td><a href="v_call/">v_call</a></td>
<td>V gene with allele</td>
</tr>
<tr>
<td><a href="v_cigar/">v_cigar</a></td>
<td>CIGAR string for the V gene alignment</td>
</tr>
<tr>
<td><a href="v_frameshift/">v_frameshift</a></td>
<td>True if the V gene in the query nucleotide sequence contains a translational ...</td>
</tr>
<tr>
<td><a href="v_gene_delineations/">v_gene_delineations</a></td>
<td></td>
</tr>
<tr>
<td><a href="v_germline_alignment/">v_germline_alignment</a></td>
<td>Aligned V gene germline sequence spanning the same region as the v_sequence_a...</td>
</tr>
<tr>
<td><a href="v_germline_alignment_aa/">v_germline_alignment_aa</a></td>
<td>Amino acid translation of the v_germline_alignment field</td>
</tr>
<tr>
<td><a href="v_germline_end/">v_germline_end</a></td>
<td>Alignment end position in the V gene reference sequence (1-based closed inter...</td>
</tr>
<tr>
<td><a href="v_germline_start/">v_germline_start</a></td>
<td>Alignment start position in the V gene reference sequence (1-based closed int...</td>
</tr>
<tr>
<td><a href="v_identity/">v_identity</a></td>
<td>Fractional identity for the V gene alignment</td>
</tr>
<tr>
<td><a href="v_rs_end/">v_rs_end</a></td>
<td>End co-ordinate in the sequence field of the V recombination site (V-genes on...</td>
</tr>
<tr>
<td><a href="v_rs_start/">v_rs_start</a></td>
<td>Start co-ordinate in the sequence field of the V recombination site (V-genes ...</td>
</tr>
<tr>
<td><a href="v_score/">v_score</a></td>
<td>Alignment score for the V gene</td>
</tr>
<tr>
<td><a href="v_sequence_alignment/">v_sequence_alignment</a></td>
<td>Aligned portion of query sequence assigned to the V gene, including any indel...</td>
</tr>
<tr>
<td><a href="v_sequence_alignment_aa/">v_sequence_alignment_aa</a></td>
<td>Amino acid translation of the v_sequence_alignment field</td>
</tr>
<tr>
<td><a href="v_sequence_end/">v_sequence_end</a></td>
<td>End position of the V gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="v_sequence_start/">v_sequence_start</a></td>
<td>Start position of the V gene in the query sequence (1-based closed interval)</td>
</tr>
<tr>
<td><a href="v_support/">v_support</a></td>
<td>V gene alignment E-value, p-value, likelihood, probability or other similar m...</td>
</tr>
<tr>
<td><a href="value/">value</a></td>
<td>The measurement result value</td>
</tr>
<tr>
<td><a href="virtual_pairing/">virtual_pairing</a></td>
<td>boolean to indicate if pairing was inferred</td>
</tr>
<tr>
<td><a href="vj_in_frame/">vj_in_frame</a></td>
<td>True if the V and J gene alignments are in-frame</td>
</tr>
<tr>
<td><a href="year/">year</a></td>
<td>The year of the issue of the journal in which a reference was published</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AgeUnit/">AgeUnit</a></td>
<td></td>
</tr>
<tr>
<td><a href="Antigen/">Antigen</a></td>
<td></td>
</tr>
<tr>
<td><a href="AntigenSourceSpecies/">AntigenSourceSpecies</a></td>
<td></td>
</tr>
<tr>
<td><a href="AntigenType/">AntigenType</a></td>
<td></td>
</tr>
<tr>
<td><a href="BiologicalSex/">BiologicalSex</a></td>
<td></td>
</tr>
<tr>
<td><a href="CellSpecies/">CellSpecies</a></td>
<td></td>
</tr>
<tr>
<td><a href="CellSubset/">CellSubset</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChainSimilarityType/">ChainSimilarityType</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChainType/">ChainType</a></td>
<td></td>
</tr>
<tr>
<td><a href="CollectionTimePointRelativeUnit/">CollectionTimePointRelativeUnit</a></td>
<td></td>
</tr>
<tr>
<td><a href="CompleteSequences/">CompleteSequences</a></td>
<td></td>
</tr>
<tr>
<td><a href="CurationalTags/">CurationalTags</a></td>
<td></td>
</tr>
<tr>
<td><a href="Derivation/">Derivation</a></td>
<td></td>
</tr>
<tr>
<td><a href="Disease/">Disease</a></td>
<td></td>
</tr>
<tr>
<td><a href="DiseaseDiagnosis/">DiseaseDiagnosis</a></td>
<td></td>
</tr>
<tr>
<td><a href="DiseaseStage/">DiseaseStage</a></td>
<td></td>
</tr>
<tr>
<td><a href="Ethnicity/">Ethnicity</a></td>
<td></td>
</tr>
<tr>
<td><a href="ExposureMaterial/">ExposureMaterial</a></td>
<td></td>
</tr>
<tr>
<td><a href="ExpressionStudyMethod/">ExpressionStudyMethod</a></td>
<td></td>
</tr>
<tr>
<td><a href="FileType/">FileType</a></td>
<td></td>
</tr>
<tr>
<td><a href="Gene/">Gene</a></td>
<td></td>
</tr>
<tr>
<td><a href="Geolocation/">Geolocation</a></td>
<td></td>
</tr>
<tr>
<td><a href="InferenceProcess/">InferenceProcess</a></td>
<td></td>
</tr>
<tr>
<td><a href="InferenceType/">InferenceType</a></td>
<td></td>
</tr>
<tr>
<td><a href="JCodonFrame/">JCodonFrame</a></td>
<td></td>
</tr>
<tr>
<td><a href="KeywordsStudy/">KeywordsStudy</a></td>
<td></td>
</tr>
<tr>
<td><a href="LibraryGenerationMethod/">LibraryGenerationMethod</a></td>
<td></td>
</tr>
<tr>
<td><a href="LifeEventProcess/">LifeEventProcess</a></td>
<td></td>
</tr>
<tr>
<td><a href="LigandType/">LigandType</a></td>
<td></td>
</tr>
<tr>
<td><a href="Locus/">Locus</a></td>
<td></td>
</tr>
<tr>
<td><a href="MhcClass/">MhcClass</a></td>
<td></td>
</tr>
<tr>
<td><a href="MhcGene1/">MhcGene1</a></td>
<td></td>
</tr>
<tr>
<td><a href="MhcGene2/">MhcGene2</a></td>
<td></td>
</tr>
<tr>
<td><a href="ObservationType/">ObservationType</a></td>
<td></td>
</tr>
<tr>
<td><a href="PairedReadDirection/">PairedReadDirection</a></td>
<td></td>
</tr>
<tr>
<td><a href="PcrTargetLocus/">PcrTargetLocus</a></td>
<td></td>
</tr>
<tr>
<td><a href="PhenotypicSex/">PhenotypicSex</a></td>
<td></td>
</tr>
<tr>
<td><a href="PhysicalLinkage/">PhysicalLinkage</a></td>
<td></td>
</tr>
<tr>
<td><a href="Property/">Property</a></td>
<td></td>
</tr>
<tr>
<td><a href="Race/">Race</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReactivityMethod/">ReactivityMethod</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReactivityReadout/">ReactivityReadout</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReadDirection/">ReadDirection</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReceptorType/">ReceptorType</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReceptorVariableDomain1Locus/">ReceptorVariableDomain1Locus</a></td>
<td></td>
</tr>
<tr>
<td><a href="ReceptorVariableDomain2Locus/">ReceptorVariableDomain2Locus</a></td>
<td></td>
</tr>
<tr>
<td><a href="SequenceType/">SequenceType</a></td>
<td></td>
</tr>
<tr>
<td><a href="Sex/">Sex</a></td>
<td></td>
</tr>
<tr>
<td><a href="Species/">Species</a></td>
<td></td>
</tr>
<tr>
<td><a href="SpeciesSubgroupType/">SpeciesSubgroupType</a></td>
<td></td>
</tr>
<tr>
<td><a href="Status/">Status</a></td>
<td></td>
</tr>
<tr>
<td><a href="Strain/">Strain</a></td>
<td></td>
</tr>
<tr>
<td><a href="Strand/">Strand</a></td>
<td></td>
</tr>
<tr>
<td><a href="StudyType/">StudyType</a></td>
<td></td>
</tr>
<tr>
<td><a href="TemplateAmountUnit/">TemplateAmountUnit</a></td>
<td></td>
</tr>
<tr>
<td><a href="TemplateClass/">TemplateClass</a></td>
<td></td>
</tr>
<tr>
<td><a href="Tissue/">Tissue</a></td>
<td></td>
</tr>
<tr>
<td><a href="Unit/">Unit</a></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>





                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
      
        
        <a href="about/" class="md-footer__link md-footer__link--next" aria-label="Next: About" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              About
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="assets/javascripts/bundle.5a2dcb6a.min.js"></script>
      
    
    
  </body>
</html>