
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://airr-knowledge.github.io/ak-schema/V1p4SampleProcessing/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-8.5.10" name="generator"/>
<title>Class: V1p4SampleProcessing - ak-schema</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-v1p4sampleprocessing">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="ak-schema" class="md-header__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            ak-schema
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: V1p4SampleProcessing
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="ak-schema" class="md-nav__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    ak-schema
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
    Usages
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/airr-knowledge/ak-schema/edit/master/docs/V1p4SampleProcessing.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-v1p4sampleprocessing">Class: V1p4SampleProcessing</h1>
<p>URI: <a href="https://github.com/airr-knowledge/ak-schema/V1p4SampleProcessing">ak_schema:V1p4SampleProcessing</a></p>
<div class="mermaid"> classDiagram
    class V1p4SampleProcessing
    click V1p4SampleProcessing href "../V1p4SampleProcessing"
      V1p4SampleProcessing : v1p4__cell_processing__cell_isolation

      V1p4SampleProcessing : v1p4__cell_processing__cell_label

      V1p4SampleProcessing : v1p4__cell_processing__cell_number

      V1p4SampleProcessing : v1p4__cell_processing__cell_phenotype

      V1p4SampleProcessing : v1p4__cell_processing__cell_processing_protocol

      V1p4SampleProcessing : v1p4__cell_processing__cell_quality

      V1p4SampleProcessing : v1p4__cell_processing__cell_species




    V1p4SampleProcessing --&gt; "0..1" V1p4CellSpecies : v1p4__cell_processing__cell_species
    click V1p4CellSpecies href "../V1p4CellSpecies"


      V1p4SampleProcessing : v1p4__cell_processing__cell_storage

      V1p4SampleProcessing : v1p4__cell_processing__cell_subset




    V1p4SampleProcessing --&gt; "1" V1p4CellSubset : v1p4__cell_processing__cell_subset
    click V1p4CellSubset href "../V1p4CellSubset"


      V1p4SampleProcessing : v1p4__cell_processing__cells_per_reaction

      V1p4SampleProcessing : v1p4__cell_processing__single_cell

      V1p4SampleProcessing : v1p4__cell_processing__tissue_processing

      V1p4SampleProcessing : v1p4__nucleic_acid_processing__complete_sequences




    V1p4SampleProcessing --&gt; "1" V1p4CompleteSequences : v1p4__nucleic_acid_processing__complete_sequences
    click V1p4CompleteSequences href "../V1p4CompleteSequences"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__library_generation_kit_version

      V1p4SampleProcessing : v1p4__nucleic_acid_processing__library_generation_method




    V1p4SampleProcessing --&gt; "1" V1p4LibraryGenerationMethod : v1p4__nucleic_acid_processing__library_generation_method
    click V1p4LibraryGenerationMethod href "../V1p4LibraryGenerationMethod"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__library_generation_protocol

      V1p4SampleProcessing : v1p4__nucleic_acid_processing__pcr_target




    V1p4SampleProcessing --&gt; "*" V1p4PCRTarget : v1p4__nucleic_acid_processing__pcr_target
    click V1p4PCRTarget href "../V1p4PCRTarget"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__physical_linkage




    V1p4SampleProcessing --&gt; "1" V1p4PhysicalLinkage : v1p4__nucleic_acid_processing__physical_linkage
    click V1p4PhysicalLinkage href "../V1p4PhysicalLinkage"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__template_amount




    V1p4SampleProcessing --&gt; "1" V1p4PhysicalQuantity : v1p4__nucleic_acid_processing__template_amount
    click V1p4PhysicalQuantity href "../V1p4PhysicalQuantity"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__template_class




    V1p4SampleProcessing --&gt; "1" V1p4TemplateClass : v1p4__nucleic_acid_processing__template_class
    click V1p4TemplateClass href "../V1p4TemplateClass"


      V1p4SampleProcessing : v1p4__nucleic_acid_processing__template_quality

      V1p4SampleProcessing : v1p4__sample__anatomic_site

      V1p4SampleProcessing : v1p4__sample__biomaterial_provider

      V1p4SampleProcessing : v1p4__sample__collection_location




    V1p4SampleProcessing --&gt; "0..1" V1p4CollectionLocation : v1p4__sample__collection_location
    click V1p4CollectionLocation href "../V1p4CollectionLocation"


      V1p4SampleProcessing : v1p4__sample__collection_time_point_relative




    V1p4SampleProcessing --&gt; "1" V1p4TimePoint : v1p4__sample__collection_time_point_relative
    click V1p4TimePoint href "../V1p4TimePoint"


      V1p4SampleProcessing : v1p4__sample__disease_state_sample

      V1p4SampleProcessing : v1p4__sample__sample_id

      V1p4SampleProcessing : v1p4__sample__sample_type

      V1p4SampleProcessing : v1p4__sample__tissue




    V1p4SampleProcessing --&gt; "1" V1p4Tissue : v1p4__sample__tissue
    click V1p4Tissue href "../V1p4Tissue"


      V1p4SampleProcessing : v1p4__sample_processing__sample_processing_id

      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_facility

      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_files




    V1p4SampleProcessing --&gt; "0..1" V1p4SequencingData : v1p4__sequencing_run__sequencing_files
    click V1p4SequencingData href "../V1p4SequencingData"


      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_kit

      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_platform

      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_run_date

      V1p4SampleProcessing : v1p4__sequencing_run__sequencing_run_id

      V1p4SampleProcessing : v1p4__sequencing_run__total_reads_passing_qc_filter


</div>
<!-- no inheritance hierarchy -->
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../v1p4__sample_processing__sample_processing_id/">v1p4__sample_processing__sample_processing_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Identifier for the sample processing object</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__sample_id/">v1p4__sample__sample_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Sample ID assigned by submitter, unique within study</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__sample_type/">v1p4__sample__sample_type</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The way the sample was obtained, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__tissue/">v1p4__sample__tissue</a></td>
<td>1 <br/> <a href="../V1p4Tissue/">V1p4Tissue</a></td>
<td>The actual tissue sampled, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__anatomic_site/">v1p4__sample__anatomic_site</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>The anatomic location of the tissue, e</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__disease_state_sample/">v1p4__sample__disease_state_sample</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Histopathologic evaluation of the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__collection_time_point_relative/">v1p4__sample__collection_time_point_relative</a></td>
<td>1 <br/> <a href="../V1p4TimePoint/">V1p4TimePoint</a></td>
<td>Time point at which sample was taken, relative to <code>label</code> event</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__collection_location/">v1p4__sample__collection_location</a></td>
<td>0..1 <br/> <a href="../V1p4CollectionLocation/">V1p4CollectionLocation</a></td>
<td>Location where the sample was taken, preferred granularity is country-level</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sample__biomaterial_provider/">v1p4__sample__biomaterial_provider</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name and address of the entity providing the sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__tissue_processing/">v1p4__cell_processing__tissue_processing</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Enzymatic digestion and/or physical methods used to isolate cells from sample</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_subset/">v1p4__cell_processing__cell_subset</a></td>
<td>1 <br/> <a href="../V1p4CellSubset/">V1p4CellSubset</a></td>
<td>Commonly-used designation of isolated cell population</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_phenotype/">v1p4__cell_processing__cell_phenotype</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>List of cellular markers and their expression levels used to isolate the cell...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_label/">v1p4__cell_processing__cell_label</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Free text cell type annotation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_species/">v1p4__cell_processing__cell_species</a></td>
<td>0..1 <br/> <a href="../V1p4CellSpecies/">V1p4CellSpecies</a></td>
<td>Binomial designation of the species from which the analyzed cells originate</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__single_cell/">v1p4__cell_processing__single_cell</a></td>
<td>1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>TRUE if single cells were isolated into separate compartments</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_number/">v1p4__cell_processing__cell_number</a></td>
<td>1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total number of cells that went into the experiment</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cells_per_reaction/">v1p4__cell_processing__cells_per_reaction</a></td>
<td>1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of cells for each biological replicate</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_storage/">v1p4__cell_processing__cell_storage</a></td>
<td>1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>TRUE if cells were cryo-preserved between isolation and further processing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_quality/">v1p4__cell_processing__cell_quality</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Relative amount of viable cells after preparation and (if applicable) thawing</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_isolation/">v1p4__cell_processing__cell_isolation</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Description of the procedure used for marker-based isolation or enrich cells</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__cell_processing__cell_processing_protocol/">v1p4__cell_processing__cell_processing_protocol</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Description of the methods applied to the sample including cell preparation/ ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__template_class/">v1p4__nucleic_acid_processing__template_class</a></td>
<td>1 <br/> <a href="../V1p4TemplateClass/">V1p4TemplateClass</a></td>
<td>The class of nucleic acid that was used as primary starting material for the ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__template_quality/">v1p4__nucleic_acid_processing__template_quality</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Description and results of the quality control performed on the template mate...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__template_amount/">v1p4__nucleic_acid_processing__template_amount</a></td>
<td>1 <br/> <a href="../V1p4PhysicalQuantity/">V1p4PhysicalQuantity</a></td>
<td>Amount of template that went into the process</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__library_generation_method/">v1p4__nucleic_acid_processing__library_generation_method</a></td>
<td>1 <br/> <a href="../V1p4LibraryGenerationMethod/">V1p4LibraryGenerationMethod</a></td>
<td>Generic type of library generation</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__library_generation_protocol/">v1p4__nucleic_acid_processing__library_generation_protocol</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Description of processes applied to substrate to obtain a library that is rea...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__library_generation_kit_version/">v1p4__nucleic_acid_processing__library_generation_kit_version</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>When using a library generation protocol from a commercial provider, provide ...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__pcr_target/">v1p4__nucleic_acid_processing__pcr_target</a></td>
<td>* <br/> <a href="../V1p4PCRTarget/">V1p4PCRTarget</a></td>
<td>If a PCR step was performed that specifically targets the IG/TR loci, the tar...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__complete_sequences/">v1p4__nucleic_acid_processing__complete_sequences</a></td>
<td>1 <br/> <a href="../V1p4CompleteSequences/">V1p4CompleteSequences</a></td>
<td>To be considered <code>complete</code>, the procedure used for library construction MUST...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__nucleic_acid_processing__physical_linkage/">v1p4__nucleic_acid_processing__physical_linkage</a></td>
<td>1 <br/> <a href="../V1p4PhysicalLinkage/">V1p4PhysicalLinkage</a></td>
<td>In case an experimental setup is used that physically links nucleic acids der...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_run_id/">v1p4__sequencing_run__sequencing_run_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>ID of sequencing run assigned by the sequencing facility</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__total_reads_passing_qc_filter/">v1p4__sequencing_run__total_reads_passing_qc_filter</a></td>
<td>1 <br/> <a href="../Integer/">Integer</a></td>
<td>Number of usable reads for analysis</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_platform/">v1p4__sequencing_run__sequencing_platform</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Designation of sequencing instrument used</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_facility/">v1p4__sequencing_run__sequencing_facility</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name and address of sequencing facility</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_run_date/">v1p4__sequencing_run__sequencing_run_date</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Date of sequencing run</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_kit/">v1p4__sequencing_run__sequencing_kit</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Name, manufacturer, order and lot numbers of sequencing kit</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v1p4__sequencing_run__sequencing_files/">v1p4__sequencing_run__sequencing_files</a></td>
<td>0..1 <br/> <a href="../V1p4SequencingData/">V1p4SequencingData</a></td>
<td>Set of sequencing files produced by the sequencing run</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../V1p4Repertoire/">V1p4Repertoire</a></td>
<td><a href="../v1p4__repertoire__sample/">v1p4__repertoire__sample</a></td>
<td>range</td>
<td><a href="./">V1p4SampleProcessing</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://github.com/airr-knowledge/ak-schema</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>ak_schema:V1p4SampleProcessing</td>
</tr>
<tr>
<td>native</td>
<td>ak_schema:V1p4SampleProcessing</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: v1p4_SampleProcessing
from_schema: https://github.com/airr-knowledge/ak-schema
slots:
- v1p4__sample_processing__sample_processing_id
- v1p4__sample__sample_id
- v1p4__sample__sample_type
- v1p4__sample__tissue
- v1p4__sample__anatomic_site
- v1p4__sample__disease_state_sample
- v1p4__sample__collection_time_point_relative
- v1p4__sample__collection_location
- v1p4__sample__biomaterial_provider
- v1p4__cell_processing__tissue_processing
- v1p4__cell_processing__cell_subset
- v1p4__cell_processing__cell_phenotype
- v1p4__cell_processing__cell_label
- v1p4__cell_processing__cell_species
- v1p4__cell_processing__single_cell
- v1p4__cell_processing__cell_number
- v1p4__cell_processing__cells_per_reaction
- v1p4__cell_processing__cell_storage
- v1p4__cell_processing__cell_quality
- v1p4__cell_processing__cell_isolation
- v1p4__cell_processing__cell_processing_protocol
- v1p4__nucleic_acid_processing__template_class
- v1p4__nucleic_acid_processing__template_quality
- v1p4__nucleic_acid_processing__template_amount
- v1p4__nucleic_acid_processing__library_generation_method
- v1p4__nucleic_acid_processing__library_generation_protocol
- v1p4__nucleic_acid_processing__library_generation_kit_version
- v1p4__nucleic_acid_processing__pcr_target
- v1p4__nucleic_acid_processing__complete_sequences
- v1p4__nucleic_acid_processing__physical_linkage
- v1p4__sequencing_run__sequencing_run_id
- v1p4__sequencing_run__total_reads_passing_qc_filter
- v1p4__sequencing_run__sequencing_platform
- v1p4__sequencing_run__sequencing_facility
- v1p4__sequencing_run__sequencing_run_date
- v1p4__sequencing_run__sequencing_kit
- v1p4__sequencing_run__sequencing_files

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: v1p4_SampleProcessing
from_schema: https://github.com/airr-knowledge/ak-schema
attributes:
  v1p4__sample_processing__sample_processing_id:
    name: v1p4__sample_processing__sample_processing_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Identifier for the sample processing object. This field should be
      unique within the repertoire. This field can be used to uniquely identify the
      combination of sample, cell processing, nucleic acid processing and sequencing
      run information for the repertoire.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample_processing__sample_processing_id
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SampleProcessing
    range: string
    required: false
  v1p4__sample__sample_id:
    name: v1p4__sample__sample_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Sample ID assigned by submitter, unique within study. If possible,
      a persistent sample ID linked to INSDC or similar repository study should be
      used.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__sample_id
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sample__sample_type:
    name: v1p4__sample__sample_type
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The way the sample was obtained, e.g. fine-needle aspirate, organ
      harvest, peripheral venous puncture
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__sample_type
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sample__tissue:
    name: v1p4__sample__tissue
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The actual tissue sampled, e.g. lymph node, liver, peripheral blood
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__tissue
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: v1p4_Tissue
    required: true
  v1p4__sample__anatomic_site:
    name: v1p4__sample__anatomic_site
    annotations:
      nullable:
        tag: nullable
        value: true
    description: The anatomic location of the tissue, e.g. Inguinal, femur
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__anatomic_site
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sample__disease_state_sample:
    name: v1p4__sample__disease_state_sample
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Histopathologic evaluation of the sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__disease_state_sample
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sample__collection_time_point_relative:
    name: v1p4__sample__collection_time_point_relative
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Time point at which sample was taken, relative to `label` event
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__collection_time_point_relative
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: v1p4_TimePoint
    required: true
  v1p4__sample__collection_location:
    name: v1p4__sample__collection_location
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Location where the sample was taken, preferred granularity is country-level
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__collection_location
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: v1p4_CollectionLocation
    required: false
  v1p4__sample__biomaterial_provider:
    name: v1p4__sample__biomaterial_provider
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name and address of the entity providing the sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sample__biomaterial_provider
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_Sample
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__cell_processing__tissue_processing:
    name: v1p4__cell_processing__tissue_processing
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Enzymatic digestion and/or physical methods used to isolate cells
      from sample
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__tissue_processing
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__cell_processing__cell_subset:
    name: v1p4__cell_processing__cell_subset
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Commonly-used designation of isolated cell population
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_subset
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: v1p4_CellSubset
    required: true
  v1p4__cell_processing__cell_phenotype:
    name: v1p4__cell_processing__cell_phenotype
    annotations:
      nullable:
        tag: nullable
        value: true
    description: List of cellular markers and their expression levels used to isolate
      the cell population.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_phenotype
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__cell_processing__cell_label:
    name: v1p4__cell_processing__cell_label
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Free text cell type annotation. Primarily used for annotating cell
      types that are not  provided in the Cell Ontology.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_label
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: false
  v1p4__cell_processing__cell_species:
    name: v1p4__cell_processing__cell_species
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Binomial designation of the species from which the analyzed cells
      originate. Typically, this value should be identical to `species`, in which
      case it SHOULD NOT be set explicitly. However, there are valid experimental
      setups in which the two might differ, e.g., chimeric animal models. If set,
      this key will overwrite the `species` information for all lower layers of the
      schema.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_species
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: v1p4_CellSpecies
    required: false
  v1p4__cell_processing__single_cell:
    name: v1p4__cell_processing__single_cell
    annotations:
      nullable:
        tag: nullable
        value: true
    description: TRUE if single cells were isolated into separate compartments
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__single_cell
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: boolean
    required: true
  v1p4__cell_processing__cell_number:
    name: v1p4__cell_processing__cell_number
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Total number of cells that went into the experiment
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_number
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: integer
    required: true
  v1p4__cell_processing__cells_per_reaction:
    name: v1p4__cell_processing__cells_per_reaction
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Number of cells for each biological replicate
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cells_per_reaction
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: integer
    required: true
  v1p4__cell_processing__cell_storage:
    name: v1p4__cell_processing__cell_storage
    annotations:
      nullable:
        tag: nullable
        value: true
    description: TRUE if cells were cryo-preserved between isolation and further processing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_storage
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: boolean
    required: true
  v1p4__cell_processing__cell_quality:
    name: v1p4__cell_processing__cell_quality
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Relative amount of viable cells after preparation and (if applicable)
      thawing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_quality
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__cell_processing__cell_isolation:
    name: v1p4__cell_processing__cell_isolation
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of the procedure used for marker-based isolation or enrich
      cells
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_isolation
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__cell_processing__cell_processing_protocol:
    name: v1p4__cell_processing__cell_processing_protocol
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of the methods applied to the sample including cell preparation/
      isolation/enrichment and nucleic acid extraction. This should closely mirror
      the Materials and methods section in the manuscript.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__cell_processing__cell_processing_protocol
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_CellProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__nucleic_acid_processing__template_class:
    name: v1p4__nucleic_acid_processing__template_class
    annotations:
      nullable:
        tag: nullable
        value: false
    description: The class of nucleic acid that was used as primary starting material
      for the following procedures
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__template_class
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_TemplateClass
    required: true
  v1p4__nucleic_acid_processing__template_quality:
    name: v1p4__nucleic_acid_processing__template_quality
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description and results of the quality control performed on the template
      material
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__template_quality
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__nucleic_acid_processing__template_amount:
    name: v1p4__nucleic_acid_processing__template_amount
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Amount of template that went into the process
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__template_amount
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_PhysicalQuantity
    required: true
  v1p4__nucleic_acid_processing__library_generation_method:
    name: v1p4__nucleic_acid_processing__library_generation_method
    annotations:
      nullable:
        tag: nullable
        value: false
    description: Generic type of library generation
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__library_generation_method
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_LibraryGenerationMethod
    required: true
  v1p4__nucleic_acid_processing__library_generation_protocol:
    name: v1p4__nucleic_acid_processing__library_generation_protocol
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Description of processes applied to substrate to obtain a library
      that is ready for sequencing
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__library_generation_protocol
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__nucleic_acid_processing__library_generation_kit_version:
    name: v1p4__nucleic_acid_processing__library_generation_kit_version
    annotations:
      nullable:
        tag: nullable
        value: true
    description: When using a library generation protocol from a commercial provider,
      provide the protocol version number
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__library_generation_kit_version
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__nucleic_acid_processing__pcr_target:
    name: v1p4__nucleic_acid_processing__pcr_target
    annotations:
      nullable:
        tag: nullable
        value: false
    description: If a PCR step was performed that specifically targets the IG/TR loci,
      the target and primer locations need to be provided here. This field holds an
      array of PCRTarget objects, so that multiplex PCR setups amplifying multiple
      loci at the same time can be annotated using one record per locus. PCR setups
      not targeting any specific locus must not annotate this field but select the
      appropriate library_generation_method instead.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__pcr_target
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_PCRTarget
    required: false
    multivalued: true
  v1p4__nucleic_acid_processing__complete_sequences:
    name: v1p4__nucleic_acid_processing__complete_sequences
    annotations:
      nullable:
        tag: nullable
        value: false
    description: To be considered `complete`, the procedure used for library construction
      MUST generate sequences that 1) include the first V gene codon that encodes
      the mature polypeptide chain (i.e. after the leader sequence) and 2) include
      the last complete codon of the J gene (i.e. 1 bp 5' of the J-&gt;C splice site)
      and 3) provide sequence information for all positions between 1) and 2). To
      be considered `complete &amp; untemplated`, the sections of the sequences defined
      in points 1) to 3) of the previous sentence MUST be untemplated, i.e. MUST NOT
      overlap with the primers used in library preparation. `mixed` should only be
      used if the procedure used for library construction will likely produce multiple
      categories of sequences in the given experiment. It SHOULD NOT be used as a
      replacement of a NULL value.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__complete_sequences
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_CompleteSequences
    required: true
  v1p4__nucleic_acid_processing__physical_linkage:
    name: v1p4__nucleic_acid_processing__physical_linkage
    annotations:
      nullable:
        tag: nullable
        value: false
    description: In case an experimental setup is used that physically links nucleic
      acids derived from distinct `Rearrangements` before library preparation, this
      field describes the mode of that linkage. All `hetero_*` terms indicate that
      in case of paired-read sequencing, the two reads should be expected to map to
      distinct IG/TR loci. `*_head-head` refers to techniques that link the 5' ends
      of transcripts in a single-cell context. `*_tail-head` refers to techniques
      that link the 3' end of one transcript to the 5' end of another one in a single-cell
      context. This term does not provide any information whether a continuous reading-frame
      between the two is generated. `*_prelinked` refers to constructs in which the
      linkage was already present on the DNA level (e.g. scFv).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__nucleic_acid_processing__physical_linkage
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_NucleicAcidProcessing
    - v1p4_SampleProcessing
    range: v1p4_PhysicalLinkage
    required: true
  v1p4__sequencing_run__sequencing_run_id:
    name: v1p4__sequencing_run__sequencing_run_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: ID of sequencing run assigned by the sequencing facility
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_run_id
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sequencing_run__total_reads_passing_qc_filter:
    name: v1p4__sequencing_run__total_reads_passing_qc_filter
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Number of usable reads for analysis
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__total_reads_passing_qc_filter
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: integer
    required: true
  v1p4__sequencing_run__sequencing_platform:
    name: v1p4__sequencing_run__sequencing_platform
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Designation of sequencing instrument used
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_platform
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sequencing_run__sequencing_facility:
    name: v1p4__sequencing_run__sequencing_facility
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name and address of sequencing facility
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_facility
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sequencing_run__sequencing_run_date:
    name: v1p4__sequencing_run__sequencing_run_date
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Date of sequencing run
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_run_date
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sequencing_run__sequencing_kit:
    name: v1p4__sequencing_run__sequencing_kit
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Name, manufacturer, order and lot numbers of sequencing kit
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_kit
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: string
    required: true
  v1p4__sequencing_run__sequencing_files:
    name: v1p4__sequencing_run__sequencing_files
    annotations:
      nullable:
        tag: nullable
        value: false
    description: Set of sequencing files produced by the sequencing run
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v1p4__sequencing_run__sequencing_files
    owner: v1p4_SampleProcessing
    domain_of:
    - v1p4_SequencingRun
    - v1p4_SampleProcessing
    range: v1p4_SequencingData
    required: false

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script><script type="module">import mermaid from "https://unpkg.com/mermaid@10.9.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>