
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://airr-knowledge.github.io/ak-schema/Clone/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-8.5.10" name="generator"/>
<title>Class: Clone - ak-schema</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-clone">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="ak-schema" class="md-header__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            ak-schema
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: Clone
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="ak-schema" class="md-nav__button md-logo" data-md-component="logo" href=".." title="ak-schema">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    ak-schema
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/airr-knowledge/ak-schema" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/airr-knowledge/ak-schema/edit/master/docs/Clone.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-clone">Class: Clone</h1>
<p>URI: <a href="https://github.com/airr-knowledge/ak-schema/Clone">ak_schema:Clone</a></p>
<div class="mermaid"> classDiagram
    class Clone
    click Clone href "../Clone"
      Clone : clone_count

      Clone : clone_id

      Clone : d_alignment_end

      Clone : d_alignment_start

      Clone : d_call

      Clone : data_processing_id

      Clone : germline_alignment

      Clone : germline_alignment_aa

      Clone : j_alignment_end

      Clone : j_alignment_start

      Clone : j_call

      Clone : junction

      Clone : junction_aa

      Clone : junction_aa_length

      Clone : junction_end

      Clone : junction_length

      Clone : junction_start

      Clone : repertoire_id

      Clone : seed_id

      Clone : sequences

      Clone : umi_count

      Clone : v_alignment_end

      Clone : v_alignment_start

      Clone : v_call


</div>
<!-- no inheritance hierarchy -->
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../clone_id/">clone_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../repertoire_id/">repertoire_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../data_processing_id/">data_processing_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../sequences/">sequences</a></td>
<td>* <br/> <a href="../String/">String</a></td>
<td>List sequence_id strings that act as keys to the Rearrangement records for me...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v_call/">v_call</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>V gene with allele</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../d_call/">d_call</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>First or only D gene with allele</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../j_call/">j_call</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>J gene with allele</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction/">junction</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction_aa/">junction_aa</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Amino acid form of the junction</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction_length/">junction_length</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction_aa_length/">junction_aa_length</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../germline_alignment/">germline_alignment</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../germline_alignment_aa/">germline_alignment_aa</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v_alignment_start/">v_alignment_start</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../v_alignment_end/">v_alignment_end</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../d_alignment_start/">d_alignment_start</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../d_alignment_end/">d_alignment_end</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../j_alignment_start/">j_alignment_start</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Start position of the J gene alignment in both the sequence_alignment and ger...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../j_alignment_end/">j_alignment_end</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>End position of the J gene alignment in both the sequence_alignment and germl...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction_start/">junction_start</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Junction region start position in the alignment (1-based closed interval)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../junction_end/">junction_end</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Junction region end position in the alignment (1-based closed interval)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../umi_count/">umi_count</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td></td>
<td>direct</td>
</tr>
<tr>
<td><a href="../clone_count/">clone_count</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Absolute count of the size (number of members) of this clone in the repertoir...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../seed_id/">seed_id</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>sequence_id of the seed sequence</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://github.com/airr-knowledge/ak-schema</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>ak_schema:Clone</td>
</tr>
<tr>
<td>native</td>
<td>ak_schema:Clone</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: Clone
from_schema: https://github.com/airr-knowledge/ak-schema
slots:
- clone_id
- repertoire_id
- data_processing_id
- sequences
- v_call
- d_call
- j_call
- junction
- junction_aa
- junction_length
- junction_aa_length
- germline_alignment
- germline_alignment_aa
- v_alignment_start
- v_alignment_end
- d_alignment_start
- d_alignment_end
- j_alignment_start
- j_alignment_end
- junction_start
- junction_end
- umi_count
- clone_count
- seed_id

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: Clone
from_schema: https://github.com/airr-knowledge/ak-schema
attributes:
  clone_id:
    name: clone_id
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: clone_id
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    - Tree
    range: string
  repertoire_id:
    name: repertoire_id
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: repertoire_id
    owner: Clone
    domain_of:
    - Repertoire
    - Rearrangement
    - Clone
    - CellExpression
    range: string
  data_processing_id:
    name: data_processing_id
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: data_processing_id
    owner: Clone
    domain_of:
    - DataProcessing
    - Alignment
    - Rearrangement
    - Clone
    - CellExpression
    range: string
  sequences:
    name: sequences
    annotations:
      nullable:
        tag: nullable
        value: true
    description: List sequence_id strings that act as keys to the Rearrangement records
      for members of the clone.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: sequences
    owner: Clone
    domain_of:
    - Clone
    range: string
    required: false
    multivalued: true
  v_call:
    name: v_call
    description: V gene with allele. If referring to a known reference sequence in
      a database the relevant gene/allele nomenclature should be followed (e.g., IGHV4-59*01
      if using IMGT/GENE-DB).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v_call
    owner: Clone
    domain_of:
    - Chain
    - Rearrangement
    - Clone
    range: string
  d_call:
    name: d_call
    description: First or only D gene with allele. If referring to a known reference
      sequence in a database the relevant gene/allele nomenclature should be followed
      (e.g., IGHD3-10*01 if using IMGT/GENE-DB).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: d_call
    owner: Clone
    domain_of:
    - Chain
    - Rearrangement
    - Clone
    range: string
  j_call:
    name: j_call
    description: J gene with allele. If referring to a known reference sequence in
      a database the relevant gene/allele nomenclature should be followed (e.g., IGHJ4*02
      if using IMGT/GENE-DB).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: j_call
    owner: Clone
    domain_of:
    - Chain
    - Rearrangement
    - Clone
    range: string
  junction:
    name: junction
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    - Node
    range: string
  junction_aa:
    name: junction_aa
    description: Amino acid form of the junction.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction_aa
    owner: Clone
    domain_of:
    - Chain
    - Rearrangement
    - Clone
    - Node
    range: string
  junction_length:
    name: junction_length
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction_length
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  junction_aa_length:
    name: junction_aa_length
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction_aa_length
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  germline_alignment:
    name: germline_alignment
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: germline_alignment
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: string
    required: true
  germline_alignment_aa:
    name: germline_alignment_aa
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: germline_alignment_aa
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: string
    required: false
  v_alignment_start:
    name: v_alignment_start
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v_alignment_start
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  v_alignment_end:
    name: v_alignment_end
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: v_alignment_end
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  d_alignment_start:
    name: d_alignment_start
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: d_alignment_start
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  d_alignment_end:
    name: d_alignment_end
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: d_alignment_end
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  j_alignment_start:
    name: j_alignment_start
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Start position of the J gene alignment in both the sequence_alignment
      and germline_alignment fields (1-based closed interval).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: j_alignment_start
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  j_alignment_end:
    name: j_alignment_end
    annotations:
      nullable:
        tag: nullable
        value: true
    description: End position of the J gene alignment in both the sequence_alignment
      and germline_alignment fields (1-based closed interval).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: j_alignment_end
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  junction_start:
    name: junction_start
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Junction region start position in the alignment (1-based closed interval).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction_start
    owner: Clone
    domain_of:
    - Clone
    range: integer
    required: false
  junction_end:
    name: junction_end
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Junction region end position in the alignment (1-based closed interval).
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: junction_end
    owner: Clone
    domain_of:
    - Clone
    range: integer
    required: false
  umi_count:
    name: umi_count
    annotations:
      nullable:
        tag: nullable
        value: true
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: umi_count
    owner: Clone
    domain_of:
    - Rearrangement
    - Clone
    range: integer
    required: false
  clone_count:
    name: clone_count
    annotations:
      nullable:
        tag: nullable
        value: true
    description: Absolute count of the size (number of members) of this clone in the
      repertoire. This could simply be the number of sequences (Rearrangement records)
      observed in this clone, the number of distinct cell barcodes (unique cell_id
      values), or a more sophisticated calculation appropriate to the experimental
      protocol. Absolute count is provided versus a frequency so that downstream analysis
      tools can perform their own normalization.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: clone_count
    owner: Clone
    domain_of:
    - Clone
    range: integer
    required: false
  seed_id:
    name: seed_id
    annotations:
      nullable:
        tag: nullable
        value: true
    description: sequence_id of the seed sequence. Empty string (or null) if there
      is no seed sequence.
    from_schema: https://github.com/airr-knowledge/ak-schema
    rank: 1000
    alias: seed_id
    owner: Clone
    domain_of:
    - Clone
    range: string
    required: false

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script><script type="module">import mermaid from "https://unpkg.com/mermaid@10.9.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>